<!DOCTYPE HTML>
<!--
	Landed by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>
	    
	Publications

		</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/jquery-ui.min.css" />
		<link rel="stylesheet" href="assets/css/jquery-ui.structure.min.css" />
		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/loyal_mods.css" />
	</head>


	
<body class="is-preload">
<div id="page-wrapper">
	<!-- Header -->
  <header id="header">
    <h1 id="logo">
      <a href="index.html"> <img id="shield" src="images/medicine.shield.small.white.png" height="40" alt=""/>Goff Lab</a>
    </h1>
    <nav id="nav">
      <ul>
        <li><a href="index.html">Home</a></li>
        <!-- <li><a href="news.html">News</a></li> -->
        <li><a href="people.html">People</a></li>
        <li><a href="#">Research</a>
            <ul>
              <li><a href="preprints.html">Preprints</a></li>
              <li><a href="publications.html">Publications</a></li>
              <li><a href="software.html">Software</a></li>
              <li><a href="datasets.html">Datasets</a></li>
            </ul>
        </li>
        <!-- <li><a href="teaching.html">Teaching</a></li> -->
        <!--
        <li><a href="#">Resources</a>
            <ul>
              <li><a href="members.html">Lab Members</a></li>
              <li><a href="links.html">Links</a></li>
            </ul>
        </li>
      -->
        <li><a href="blog.html">Blog</a></li>
        <li><a href="join.html">Opportunites</a></li>
        <li><a href="contact.html">Contact Us</a></li>
        <!--<li><a href="#" class="button primary">Sign Up</a></li>-->
      </ul>
    </nav>
  </header>
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Publications</h2>
						<!-- Publication summary links -->
						<p>
							<a href="http://europepmc.org/authors/0000-0003-2875-451X" target="_blank">EBI Summary page</a> |
							<a href="https://scholar.google.com/citations?user=2ZVv2v8AAAAJ&hl=en" target="_blank">Google Scholar</a> |
							<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Goff%2C+Loyal+%5BAuthor%5D+or+Goff%2C+Loyal+Andrew%5BAuthor%5D" target="_blank">Pubmed</a> |
							<a href="https://orcid.org/0000-0003-2875-451X" target="_blank">ORCHID: 0000-0003-2875-451X</a> |
							<a href="https://profiles.impactstory.org/u/0000-0003-2875-451X" target="_blank">Impactstory</a>
						</p>
				</header>
				<!-- Content -->
					<section id="content">

							<div class="accordion alt" id="accordionpubs">
						    
								<!--  Card 1  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading1">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse1" aria-expanded="true" aria-controls="collapse1">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">1. Parallel Social Information Processing Circuits Are Differentially Impacted in Autism. (2020)</p>
						                 <p class="font-italic font-weight-light">Lewis EM, Stein-O'Brien GL, Patino AV, Nardou R, Grossman CD, Brown M, Bangamwabo B, Ndiaye N, Giovinazzo D, Dardani I, Jiang C, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Dolen G</p>
						                 <p>Neuron. 2020 Oct 21. pii: S0896-6273(20)30770-4. doi: 10.1016/j.neuron.2020.10.002.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse1" class="collapse" aria-labelledby="heading1" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33113347" class="altmetric-embed pull-left"> </div>
						          <p>Parallel processing circuits are thought to dramatically expand the network capabilities of the nervous system. Magnocellular and parvocellular oxytocin neurons have been proposed to subserve two parallel streams of social information processing, which allow a single molecule to encode a diverse array of ethologically distinct behaviors. Here we provide the first comprehensive characterization of magnocellular and parvocellular oxytocin neurons in male mice, validated across anatomical, projection target, electrophysiological, and transcriptional criteria. We next use novel multiple feature selection tools in Fmr1-KO mice to provide direct evidence that normal functioning of the parvocellular but not magnocellular oxytocin pathway is required for autism-relevant social reward behavior. Finally, we demonstrate that autism risk genes are enriched in parvocellular compared with magnocellular oxytocin neurons. Taken together, these results provide the first evidence that oxytocin-pathway-specific pathogenic mechanisms account for social impairments across a broad range of autism etiologies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S0896-6273(20)30770-4 [pii] 10.1016/j.neuron.2020.10.002 ">10.1016/j.neuron.2020.10.002 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 2  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading2">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse2" aria-expanded="true" aria-controls="collapse2">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">2. Developmental, cellular, and behavioral phenotypes in a mouse model of congenital hypoplasia of the dentate gyrus. (2020)</p>
						                 <p class="font-italic font-weight-light">Rattner A, Terrillion CE, Jou C, Kleven T, Hu SF, Williams J, Hou Z, Aggarwal M, Mori S, Shin G, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Witter MP, Pletnikov M, Fenton AA, Nathans J</p>
						                 <p>Elife. 2020 Oct 21;9. pii: 62766. doi: 10.7554/eLife.62766.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse2" class="collapse" aria-labelledby="heading2" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="33084572" class="altmetric-embed pull-left"> </div>
						          <p>In the hippocampus, a widely accepted model posits that the dentate gyrus improves learning and memory by enhancing discrimination between inputs. To test this model, we studied conditional knockout mice in which the vast majority of dentate granule cells (DGCs) fail to develop - including nearly all DGCs in the dorsal hippocampus - secondary to eliminating Wntless (Wls) in a subset of cortical progenitors with Gfap-Cre. Other cells in the Wls(fl/-);Gfap-Cre hippocampus were minimally affected, as determined by single nucleus RNA sequencing. CA3 pyramidal cells, the targets of DGC-derived mossy fibers, exhibited normal morphologies with a small reduction in the numbers of synaptic spines. Wls(fl/-);Gfap-Cre mice have a modest performance decrement in several complex spatial tasks, including active place avoidance. They were also modestly impaired in one simpler spatial task, finding a visible platform in the Morris water maze. These experiments support a role for DGCs in enhancing spatial learning and memory.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.7554/eLife.62766 ">10.7554/eLife.62766 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 3  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading3">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse3" aria-expanded="true" aria-controls="collapse3">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">3. Screening non-MAPT genes of the Chr17q21 H1 haplotype in Parkinson's disease. (2020)</p>
						                 <p class="font-italic font-weight-light">Soto-Beasley AI, Walton RL, Valentino RR, Hook PW, Labbe C, Heckman MG, Johnson PW, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Uitti RJ, McLean PJ, Springer W, McCallion AS, Wszolek ZK, Ross OA</p>
						                 <p>Parkinsonism Relat Disord. 2020 Aug 1;78:138-144. doi: 10.1016/j.parkreldis.2020.07.022.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse3" class="collapse" aria-labelledby="heading3" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32829096" class="altmetric-embed pull-left"> </div>
						          <p>INTRODUCTION: The microtubule-associated protein tau (MAPT) gene is considered a strong genetic risk factor for Parkinson's disease (PD) in Caucasians. MAPT is located within an inversion region of high linkage disequilibrium designated as H1 and H2 haplotype, and contains eight other genes which have been implicated in neurodegeneration. The aim of the current study was to identify common coding variants in strong linkage disequilibrium (LD) within the associated loci on chr17q21 harboring MAPT. METHODS: Sanger sequencing of coding exons in 90 Caucasian late-onset PD (LOPD) patients was performed. Specific gene sequencing for LRRC37A, LRRC37A2, ARL17A and ARL17B was not possible given the high homology, presence of pseudogenes and copy number variants that are in the region, and therefore four genes (NSF, KANSL1, SPPL2C, and CRHR1) were included in the analysis. Coding variants from these four genes that did not perfectly tag (r(2) = 1) the MAPT H1/H2 haplotype were genotyped in an independent replication series of Caucasian PD cases (N = 851) and controls (N = 730). RESULTS: In the 90 LOPD cases we identified 30 coding variants. Eleven non-synonymous variants tagged the MAPT H1/H2 haplotype, including two SPPL2C variants (rs12185233 and rs12373123) that had high pathogenic combined annotation dependent depletion (CADD) scores of >20. In the replication series, the non-synonymous KANSL1 rs17585974 variant was in very strong LD with MAPT H1/H2 and had a high CADD score of 24.7. CONCLUSION: We have identified several non-synonymous variants across neighboring genes of MAPT that may warrant further genetic and functional investigation within the biological etiology of PD.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S1353-8020(20)30659-3 [pii] 10.1016/j.parkreldis.2020.07.022 ">10.1016/j.parkreldis.2020.07.022 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 4  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading4">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse4" aria-expanded="true" aria-controls="collapse4">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">4. Single-Cell Analysis of Human Retina Identifies Evolutionarily Conserved and Species-Specific Mechanisms Controlling Development. (2020)</p>
						                 <p class="font-italic font-weight-light">Lu Y, Shiau F, Yi W, Lu S, Wu Q, Pearson JD, Kallman A, Zhong S, Hoang T, Zuo Z, Zhao F, Zhang M, Tsai N, Zhuo Y, He S, Zhang J, Stein-O'Brien GL, Sherman TD, Duan X, Fertig EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Zack DJ, Handa JT, Xue T, Bremner R, Blackshaw S, Wang X, Clark BS</p>
						                 <p>Dev Cell. 2020 May 18;53(4):473-491.e9. doi: 10.1016/j.devcel.2020.04.009. Epub 2020 May 7.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse4" class="collapse" aria-labelledby="heading4" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32386599" class="altmetric-embed pull-left"> </div>
						          <p>The development of single-cell RNA sequencing (scRNA-seq) has allowed high-resolution analysis of cell-type diversity and transcriptional networks controlling cell-fate specification. To identify the transcriptional networks governing human retinal development, we performed scRNA-seq analysis on 16 time points from developing retina as well as four early stages of retinal organoid differentiation. We identified evolutionarily conserved patterns of gene expression during retinal progenitor maturation and specification of all seven major retinal cell types. Furthermore, we identified gene-expression differences between developing macula and periphery and between distinct populations of horizontal cells. We also identified species-specific patterns of gene expression during human and mouse retinal development. Finally, we identified an unexpected role for ATOH7 expression in regulation of photoreceptor specification during late retinogenesis. These results provide a roadmap to future studies of human retinal development and may help guide the design of cell-based therapies for treating retinal dystrophies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S1534-5807(20)30307-5 [pii] 10.1016/j.devcel.2020.04.009 ">10.1016/j.devcel.2020.04.009 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 5  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading5">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse5" aria-expanded="true" aria-controls="collapse5">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">5. Mitoregulin Controls beta-Oxidation in Human and Mouse Adipocytes. (2020)</p>
						                 <p class="font-italic font-weight-light">Friesen M, Warren CR, Yu H, Toyohara T, Ding Q, Florido MHC, Sayre C, Pope BD, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL, Cowan CA</p>
						                 <p>Stem Cell Reports. 2020 Apr 14;14(4):590-602. doi: 10.1016/j.stemcr.2020.03.002. Epub 2020 Apr 2.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse5" class="collapse" aria-labelledby="heading5" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32243843" class="altmetric-embed pull-left"> </div>
						          <p>We previously discovered in mouse adipocytes an lncRNA (the homolog of human LINC00116) regulating adipogenesis that contains a highly conserved coding region. Here, we show human protein expression of a peptide within LINC00116, and demonstrate that this peptide modulates triglyceride clearance in human adipocytes by regulating lipolysis and mitochondrial beta-oxidation. This gene has previously been identified as mitoregulin (MTLN). We conclude that MTLN has a regulatory role in adipocyte metabolism as demonstrated by systemic lipid phenotypes in knockout mice. We also assert its adipocyte-autonomous phenotypes in both isolated murine adipocytes as well as human stem cell-derived adipocytes. MTLN directly interacts with the beta subunit of the mitochondrial trifunctional protein, an enzyme critical in the beta-oxidation of long-chain fatty acids. Our human and murine models contend that MTLN could be an avenue for further therapeutic research, albeit not without caveats, for example, by promoting white adipocyte triglyceride clearance in obese subjects.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S2213-6711(20)30070-9 [pii] 10.1016/j.stemcr.2020.03.002 ">10.1016/j.stemcr.2020.03.002 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 6  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading6">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse6" aria-expanded="true" aria-controls="collapse6">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">6. projectR: an R/Bioconductor package for transfer learning via PCA, NMF, correlation and clustering. (2020)</p>
						                 <p class="font-italic font-weight-light">Sharma G, Colantuoni C, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Fertig EJ, Stein-O'Brien G</p>
						                 <p>Bioinformatics. 2020 Jun 1;36(11):3592-3593. doi: 10.1093/bioinformatics/btaa183.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse6" class="collapse" aria-labelledby="heading6" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="32167521" class="altmetric-embed pull-left"> </div>
						          <p>MOTIVATION: Dimension reduction techniques are widely used to interpret high-dimensional biological data. Features learned from these methods are used to discover both technical artifacts and novel biological phenomena. Such feature discovery is critically importent in analysis of large single-cell datasets, where lack of a ground truth limits validation and interpretation. Transfer learning (TL) can be used to relate the features learned from one source dataset to a new target dataset to perform biologically driven validation by evaluating their use in or association with additional sample annotations in that independent target dataset. RESULTS: We developed an R/Bioconductor package, projectR, to perform TL for analyses of genomics data via TL of clustering, correlation and factorization methods. We then demonstrate the utility TL for integrated data analysis with an example for spatial single-cell analysis. AVAILABILITY AND IMPLEMENTATION: projectR is available on Bioconductor and at https://github.com/genesofeve/projectR. CONTACT: gsteinobrien@jhmi.edu or ejfertig@jhmi.edu. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btaa183 ">10.1093/bioinformatics/btaa183 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 7  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading7">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse7" aria-expanded="true" aria-controls="collapse7">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">7. Comprehensive analysis of a mouse model of spontaneous uveoretinitis using single-cell RNA sequencing. (2019)</p>
						                 <p class="font-italic font-weight-light">Heng JS, Hackett SF, Stein-O'Brien GL, Winer BL, Williams J, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Nathans J</p>
						                 <p>Proc Natl Acad Sci U S A. 2019 Dec 16. pii: 1915571116. doi: 10.1073/pnas.1915571116.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse7" class="collapse" aria-labelledby="heading7" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31843893" class="altmetric-embed pull-left"> </div>
						          <p>Autoimmune uveoretinitis is a significant cause of visual loss, and mouse models offer unique opportunities to study its disease mechanisms. Aire (-/-) mice fail to express self-antigens in the thymus, exhibit reduced central tolerance, and develop a spontaneous, chronic, and progressive uveoretinitis. Using single-cell RNA sequencing (scRNA-seq), we characterized wild-type and Aire (-/-) retinas to define, in a comprehensive and unbiased manner, the cell populations and gene expression patterns associated with disease. Based on scRNA-seq, immunostaining, and in situ hybridization, we infer that 1) the dominant effector response in Aire (-/-) retinas is Th1-driven, 2) a subset of monocytes convert to either a macrophage/microglia state or a dendritic cell state, 3) the development of tertiary lymphoid structures constitutes part of the Aire (-/-) retinal phenotype, 4) all major resident retinal cell types respond to interferon gamma (IFNG) by changing their patterns of gene expression, and 5) Muller glia up-regulate specific genes in response to IFN gamma and may act as antigen-presenting cells.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/201915571 [pii] 10.1073/pnas.1915571116 ">10.1073/pnas.1915571116 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 8  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading8">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse8" aria-expanded="true" aria-controls="collapse8">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">8. Increased expression of anion transporter SLC26A9 delays diabetes onset in cystic fibrosis. (2020)</p>
						                 <p class="font-italic font-weight-light">Lam AN, Aksit MA, Vecchio-Pagan B, Shelton CA, Osorio DL, Anzmann AF, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Whitcomb DC, Blackman SM, Cutting GR</p>
						                 <p>J Clin Invest. 2020 Jan 2;130(1):272-286. doi: 10.1172/JCI129833.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse8" class="collapse" aria-labelledby="heading8" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31581148" class="altmetric-embed pull-left"> </div>
						          <p>Diabetes is a common complication of cystic fibrosis (CF) that affects approximately 20% of adolescents and 40%-50% of adults with CF. The age at onset of CF-related diabetes (CFRD) (marked by clinical diagnosis and treatment initiation) is an important measure of the disease process. DNA variants associated with age at onset of CFRD reside in and near SLC26A9. Deep sequencing of the SLC26A9 gene in 762 individuals with CF revealed that 2 common DNA haplotypes formed by the risk variants account for the association with diabetes. Single-cell RNA sequencing (scRNA-Seq) indicated that SLC26A9 is predominantly expressed in pancreatic ductal cells and frequently coexpressed with CF transmembrane conductance regulator (CFTR) along with transcription factors that have binding sites 5' of SLC26A9. These findings were replicated upon reanalysis of scRNA-Seq data from 4 independent studies. DNA fragments derived from the 5' region of SLC26A9-bearing variants from the low-risk haplotype generated 12%-20% higher levels of expression in PANC-1 and CFPAC-1 cells compared with the high- risk haplotype. Taken together, our findings indicate that an increase in SLC26A9 expression in ductal cells of the pancreas delays the age at onset of diabetes, suggesting a CFTR-agnostic treatment for a major complication of CF.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1172/JCI129833 ">10.1172/JCI129833 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 9  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading9">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse9" aria-expanded="true" aria-controls="collapse9">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">9. A screen of 1,049 schizophrenia and 30 Alzheimer's-associated variants for regulatory potential. (2020)</p>
						                 <p class="font-italic font-weight-light">Myint L, Wang R, Boukas L, Hansen KD, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Avramopoulos D</p>
						                 <p>Am J Med Genet B Neuropsychiatr Genet. 2020 Jan;183(1):61-73. doi: 10.1002/ajmg.b.32761. Epub 2019 Sep 10.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse9" class="collapse" aria-labelledby="heading9" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31503409" class="altmetric-embed pull-left"> </div>
						          <p>Recent genome-wide association studies (GWAS) identified numerous schizophrenia (SZ) and Alzheimer's disease (AD) associated loci, most outside protein-coding regions and hypothesized to affect gene transcription. We used a massively parallel reporter assay to screen, 1,049 SZ and 30 AD variants in 64 and nine loci, respectively for allele differences in driving reporter gene expression. A library of synthetic oligonucleotides assaying each allele five times was transfected into K562 chronic myelogenous leukemia lymphoblasts and SK-SY5Y human neuroblastoma cells. One hundred forty eight variants showed allelic differences in K562 and 53 in SK-SY5Y cells, on average 2.6 variants per locus. Nine showed significant differences in both lines, a modest overlap reflecting different regulatory landscapes of these lines that also differ significantly in chromatin marks. Eight of nine were in the same direction. We observe no preference for risk alleles to increase or decrease expression. We find a positive correlation between the number of SNPs in linkage disequilibrium and the proportion of functional SNPs supporting combinatorial effects that may lead to haplotype selection. Our results prioritize future functional follow up of disease associated SNPs to determine the driver GWAS variant(s), at each locus and enhance our understanding of gene regulation dynamics.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1002/ajmg.b.32761 ">10.1002/ajmg.b.32761 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 10  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading10">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse10" aria-expanded="true" aria-controls="collapse10">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">10. Precocious neuronal differentiation and disrupted oxygen responses in Kabuki syndrome. (2019)</p>
						                 <p class="font-italic font-weight-light">Carosso GA, Boukas L, Augustin JJ, Nguyen HN, Winer BL, Cannon GH, Robertson JD, Zhang L, Hansen KD, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Bjornsson HT</p>
						                 <p>JCI Insight. 2019 Oct 17;4(20). pii: 129375. doi: 10.1172/jci.insight.129375.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse10" class="collapse" aria-labelledby="heading10" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31465303" class="altmetric-embed pull-left"> </div>
						          <p>Chromatin modifiers act to coordinate gene expression changes critical to neuronal differentiation from neural stem/progenitor cells (NSPCs). Lysine-specific methyltransferase 2D (KMT2D) encodes a histone methyltransferase that promotes transcriptional activation and is frequently mutated in cancers and in the majority (>70%) of patients diagnosed with the congenital, multisystem intellectual disability disorder Kabuki syndrome 1 (KS1). Critical roles for KMT2D are established in various non-neural tissues, but the effects of KMT2D loss in brain cell development have not been described. We conducted parallel studies of proliferation, differentiation, transcription, and chromatin profiling in KMT2D-deficient human and mouse models to define KMT2D-regulated functions in neurodevelopmental contexts, including adult-born hippocampal NSPCs in vivo and in vitro. We report cell-autonomous defects in proliferation, cell cycle, and survival, accompanied by early NSPC maturation in several KMT2D-deficient model systems. Transcriptional suppression in KMT2D-deficient cells indicated strong perturbation of hypoxia-responsive metabolism pathways. Functional experiments confirmed abnormalities of cellular hypoxia responses in KMT2D-deficient neural cells and accelerated NSPC maturation in vivo. Together, our findings support a model in which loss of KMT2D function suppresses expression of oxygen-responsive gene programs important to neural progenitor maintenance, resulting in precocious neuronal differentiation in a mouse model of KS1.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1172/jci.insight.129375 ">10.1172/jci.insight.129375 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 11  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading11">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse11" aria-expanded="true" aria-controls="collapse11">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">11. Differential Variation Analysis Enables Detection of Tumor Heterogeneity Using Single-Cell RNA-Sequencing Data. (2019)</p>
						                 <p class="font-italic font-weight-light">Davis-Marcisak EF, Sherman TD, Orugunta P, Stein-O'Brien GL, Puram SV, Roussos Torres ET, Hopkins AC, Jaffee EM, Favorov AV, Afsari B, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Fertig EJ</p>
						                 <p>Cancer Res. 2019 Oct 1;79(19):5102-5112. doi: 10.1158/0008-5472.CAN-18-3882. Epub 2019 Jul 23.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse11" class="collapse" aria-labelledby="heading11" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31337651" class="altmetric-embed pull-left"> </div>
						          <p>Tumor heterogeneity provides a complex challenge to cancer treatment and is a critical component of therapeutic response, disease recurrence, and patient survival. Single-cell RNA-sequencing (scRNA-seq) technologies have revealed the prevalence of intratumor and intertumor heterogeneity. Computational techniques are essential to quantify the differences in variation of these profiles between distinct cell types, tumor subtypes, and patients to fully characterize intratumor and intertumor molecular heterogeneity. In this study, we adapted our algorithm for pathway dysregulation, Expression Variation Analysis (EVA), to perform multivariate statistical analyses of differential variation of expression in gene sets for scRNA-seq. EVA has high sensitivity and specificity to detect pathways with true differential heterogeneity in simulated data. EVA was applied to several public domain scRNA-seq tumor datasets to quantify the landscape of tumor heterogeneity in several key applications in cancer genomics such as immunogenicity, metastasis, and cancer subtypes. Immune pathway heterogeneity of hematopoietic cell populations in breast tumors corresponded to the amount of diversity present in the T-cell repertoire of each individual. Cells from head and neck squamous cell carcinoma (HNSCC) primary tumors had significantly more heterogeneity across pathways than cells from metastases, consistent with a model of clonal outgrowth. Moreover, there were dramatic differences in pathway dysregulation across HNSCC basal primary tumors. Within the basal primary tumors, there was increased immune dysregulation in individuals with a high proportion of fibroblasts present in the tumor microenvironment. These results demonstrate the broad utility of EVA to quantify intertumor and intratumor heterogeneity from scRNA-seq data without reliance on low-dimensional visualization. SIGNIFICANCE: This study presents a robust statistical algorithm for evaluating gene expression heterogeneity within pathways or gene sets in single-cell RNA-seq data.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1158/0008-5472.CAN-18-3882 ">10.1158/0008-5472.CAN-18-3882 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 12  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading12">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse12" aria-expanded="true" aria-controls="collapse12">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">12. Single-Cell RNA-Seq Analysis of Retinal Development Identifies NFI Factors as Regulating Mitotic Exit and Late-Born Cell Specification. (2019)</p>
						                 <p class="font-italic font-weight-light">Clark BS, Stein-O'Brien GL, Shiau F, Cannon GH, Davis-Marcisak E, Sherman T, Santiago CP, Hoang TV, Rajaii F, James-Esposito RE, Gronostajski RM, Fertig EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Blackshaw S</p>
						                 <p>Neuron. 2019 Jun 19;102(6):1111-1126.e5. doi: 10.1016/j.neuron.2019.04.010. Epub 2019 May 22.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse12" class="collapse" aria-labelledby="heading12" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31128945" class="altmetric-embed pull-left"> </div>
						          <p>Precise temporal control of gene expression in neuronal progenitors is necessary for correct regulation of neurogenesis and cell fate specification. However, the cellular heterogeneity of the developing CNS has posed a major obstacle to identifying the gene regulatory networks that control these processes. To address this, we used single-cell RNA sequencing to profile ten developmental stages encompassing the full course of retinal neurogenesis. This allowed us to comprehensively characterize changes in gene expression that occur during initiation of neurogenesis, changes in developmental competence, and specification and differentiation of each major retinal cell type. We identify the NFI transcription factors (Nfia, Nfib, and Nfix) as selectively expressed in late retinal progenitor cells and show that they control bipolar interneuron and Muller glia cell fate specification and promote proliferative quiescence.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S0896-6273(19)30345-9 [pii] 10.1016/j.neuron.2019.04.010 ">10.1016/j.neuron.2019.04.010 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 13  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading13">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse13" aria-expanded="true" aria-controls="collapse13">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">13. Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species. (2019)</p>
						                 <p class="font-italic font-weight-light">Stein-O'Brien GL, Clark BS, Sherman T, Zibetti C, Hu Q, Sealfon R, Liu S, Qian J, Colantuoni C, Blackshaw S, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Fertig EJ</p>
						                 <p>Cell Syst. 2019 May 22;8(5):395-411.e8. doi: 10.1016/j.cels.2019.04.004.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse13" class="collapse" aria-labelledby="heading13" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="31121116" class="altmetric-embed pull-left"> </div>
						          <p>Analysis of gene expression in single cells allows for decomposition of cellular states as low-dimensional latent spaces. However, the interpretation and validation of these spaces remains a challenge. Here, we present scCoGAPS, which defines latent spaces from a source single-cell RNA-sequencing (scRNA-seq) dataset, and projectR, which evaluates these latent spaces in independent target datasets via transfer learning. Application of developing mouse retina to scRNA-Seq reveals intrinsic relationships across biological contexts and assays while avoiding batch effects and other technical features. We compare the dimensions learned in this source dataset to adult mouse retina, a time-course of human retinal development, select scRNA-seq datasets from developing brain, chromatin accessibility data, and a murine-cell type atlas to identify shared biological features. These tools lay the groundwork for exploratory analysis of scRNA-seq data via latent space representations, enabling a shift in how we compare and identify cells beyond reliance on marker genes or ensemble molecular identity.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S2405-4712(19)30146-2 [pii] 10.1016/j.cels.2019.04.004 ">10.1016/j.cels.2019.04.004 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 14  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading14">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse14" aria-expanded="true" aria-controls="collapse14">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">14. Hypoxia tolerance in the Norrin-deficient retina and the chronically hypoxic brain studied at single-cell resolution. (2019)</p>
						                 <p class="font-italic font-weight-light">Heng JS, Rattner A, Stein-O'Brien GL, Winer BL, Jones BW, Vernon HJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Nathans J</p>
						                 <p>Proc Natl Acad Sci U S A. 2019 Apr 30;116(18):9103-9114. doi: 10.1073/pnas.1821122116. Epub 2019 Apr 15.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse14" class="collapse" aria-labelledby="heading14" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30988181" class="altmetric-embed pull-left"> </div>
						          <p>The mammalian CNS is capable of tolerating chronic hypoxia, but cell type-specific responses to this stress have not been systematically characterized. In the Norrin KO (Ndp (KO) ) mouse, a model of familial exudative vitreoretinopathy (FEVR), developmental hypovascularization of the retina produces chronic hypoxia of inner nuclear-layer (INL) neurons and Muller glia. We used single-cell RNA sequencing, untargeted metabolomics, and metabolite labeling from (13)C-glucose to compare WT and Ndp (KO) retinas. In Ndp (KO) retinas, we observe gene expression responses consistent with hypoxia in Muller glia and retinal neurons, and we find a metabolic shift that combines reduced flux through the TCA cycle with increased synthesis of serine, glycine, and glutathione. We also used single-cell RNA sequencing to compare the responses of individual cell types in Ndp (KO) retinas with those in the hypoxic cerebral cortex of mice that were housed for 1 week in a reduced oxygen environment (7.5% oxygen). In the hypoxic cerebral cortex, glial transcriptome responses most closely resemble the response of Muller glia in the Ndp (KO) retina. In both retina and brain, vascular endothelial cells activate a previously dormant tip cell gene expression program, which likely underlies the adaptive neoangiogenic response to chronic hypoxia. These analyses of retina and brain transcriptomes at single-cell resolution reveal both shared and cell type-specific changes in gene expression in response to chronic hypoxia, implying both shared and distinct cell type-specific physiologic responses.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1821122116 ">10.1073/pnas.1821122116 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 15  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading15">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse15" aria-expanded="true" aria-controls="collapse15">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">15. Linear models enable powerful differential activity analysis in massively parallel reporter assays. (2019)</p>
						                 <p class="font-italic font-weight-light">Myint L, Avramopoulos DG, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Hansen KD</p>
						                 <p>BMC Genomics. 2019 Mar 12;20(1):209. doi: 10.1186/s12864-019-5556-x.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse15" class="collapse" aria-labelledby="heading15" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30866806" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: Massively parallel reporter assays (MPRAs) have emerged as a popular means for understanding noncoding variation in a variety of conditions. While a large number of experiments have been described in the literature, analysis typically uses ad-hoc methods. There has been little attention to comparing performance of methods across datasets. RESULTS: We present the mpralm method which we show is calibrated and powerful, by analyzing its performance on multiple MPRA datasets. We show that it outperforms existing statistical methods for analysis of this data type, in the first comprehensive evaluation of statistical methods on several datasets. We investigate theoretical and real-data properties of barcode summarization methods and show an unappreciated impact of summarization method for some datasets. Finally, we use our model to conduct a power analysis for this assay and show substantial improvements in power by performing up to 6 replicates per condition, whereas sequencing depth has smaller impact; we recommend to always use at least 4 replicates. An R package is available from the Bioconductor project. CONCLUSIONS: Together, these results inform recommendations for differential analysis, general group comparisons, and power analysis and will help improve design and analysis of MPRA experiments.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/s12864-019-5556-x ">10.1186/s12864-019-5556-x </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 16  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading16">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse16" aria-expanded="true" aria-controls="collapse16">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">16. Transcriptional and epigenomic landscapes of CNS and non-CNS vascular endothelial cells. (2018)</p>
						                 <p class="font-italic font-weight-light">Sabbagh MF, Heng JS, Luo C, Castanon RG, Nery JR, Rattner A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Ecker JR, Nathans J</p>
						                 <p>Elife. 2018 Sep 6;7. pii: 36187. doi: 10.7554/eLife.36187.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse16" class="collapse" aria-labelledby="heading16" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30188322" class="altmetric-embed pull-left"> </div>
						          <p>Vascular endothelial cell (EC) function depends on appropriate organ-specific molecular and cellular specializations. To explore genomic mechanisms that control this specialization, we have analyzed and compared the transcriptome, accessible chromatin, and DNA methylome landscapes from mouse brain, liver, lung, and kidney ECs. Analysis of transcription factor (TF) gene expression and TF motifs at candidate cis-regulatory elements reveals both shared and organ-specific EC regulatory networks. In the embryo, only those ECs that are adjacent to or within the central nervous system (CNS) exhibit canonical Wnt signaling, which correlates precisely with blood-brain barrier (BBB) differentiation and Zic3 expression. In the early postnatal brain, single-cell RNA-seq of purified ECs reveals (1) close relationships between veins and mitotic cells and between arteries and tip cells, (2) a division of capillary ECs into vein-like and artery-like classes, and (3) new endothelial subtype markers, including new validated tip cell markers.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.7554/eLife.36187 ">10.7554/eLife.36187 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 17  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading17">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse17" aria-expanded="true" aria-controls="collapse17">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">17. Enter the Matrix: Factorization Uncovers Knowledge from Omics. (2018)</p>
						                 <p class="font-italic font-weight-light">Stein-O'Brien GL, Arora R, Culhane AC, Favorov AV, Garmire LX, Greene CS, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Li Y, Ngom A, Ochs MF, Xu Y, Fertig EJ</p>
						                 <p>Trends Genet. 2018 Oct;34(10):790-805. doi: 10.1016/j.tig.2018.07.003. Epub 2018 Aug 22.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse17" class="collapse" aria-labelledby="heading17" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="30143323" class="altmetric-embed pull-left"> </div>
						          <p>Omics data contain signals from the molecular, physical, and kinetic inter- and intracellular interactions that control biological systems. Matrix factorization (MF) techniques can reveal low-dimensional structure from high-dimensional data that reflect these interactions. These techniques can uncover new biological knowledge from diverse high-throughput omics data in applications ranging from pathway discovery to timecourse analysis. We review exemplary applications of MF for systems-level analyses. We discuss appropriate applications of these methods, their limitations, and focus on the analysis of results to facilitate optimal biological interpretation. The inference of biologically relevant features with MF enables discovery from high-throughput data beyond the limits of current biological knowledge - answering questions from high-dimensional data that we have not yet thought to ask.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S0168-9525(18)30124-0 [pii] 10.1016/j.tig.2018.07.003 ">10.1016/j.tig.2018.07.003 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 18  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading18">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse18" aria-expanded="true" aria-controls="collapse18">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">18. Single-Cell RNA-Seq of Mouse Dopaminergic Neurons Informs Candidate Gene Selection for Sporadic Parkinson Disease. (2018)</p>
						                 <p class="font-italic font-weight-light">Hook PW, McClymont SA, Cannon GH, Law WD, Morton AJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, McCallion AS</p>
						                 <p>Am J Hum Genet. 2018 Mar 1;102(3):427-446. doi: 10.1016/j.ajhg.2018.02.001.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse18" class="collapse" aria-labelledby="heading18" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="29499164" class="altmetric-embed pull-left"> </div>
						          <p>Genetic variation modulating risk of sporadic Parkinson disease (PD) has been primarily explored through genome-wide association studies (GWASs). However, like many other common genetic diseases, the impacted genes remain largely unknown. Here, we used single-cell RNA-seq to characterize dopaminergic (DA) neuron populations in the mouse brain at embryonic and early postnatal time points. These data facilitated unbiased identification of DA neuron subpopulations through their unique transcriptional profiles, including a postnatal neuroblast population and substantia nigra (SN) DA neurons. We use these population-specific data to develop a scoring system to prioritize candidate genes in all 49 GWAS intervals implicated in PD risk, including genes with known PD associations and many with extensive supporting literature. As proof of principle, we confirm that the nigrostriatal pathway is compromised in Cplx1-null mice. Ultimately, this systematic approach establishes biologically pertinent candidates and testable hypotheses for sporadic PD, informing a new era of PD genetic research.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S0002-9297(18)30046-6 [pii] 10.1016/j.ajhg.2018.02.001 ">10.1016/j.ajhg.2018.02.001 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 19  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading19">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse19" aria-expanded="true" aria-controls="collapse19">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">19. Variation in Activity State, Axonal Projection, and Position Define the Transcriptional Identity of Individual Neocortical Projection Neurons. (2018)</p>
						                 <p class="font-italic font-weight-light">Chevee M, Robertson JJ, Cannon GH, Brown SP, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>,</p>
						                 <p>Cell Rep. 2018 Jan 9;22(2):441-455. doi: 10.1016/j.celrep.2017.12.046.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse19" class="collapse" aria-labelledby="heading19" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="29320739" class="altmetric-embed pull-left"> </div>
						          <p>Single-cell RNA sequencing has generated catalogs of transcriptionally defined neuronal subtypes of the brain. However, the cellular processes that contribute to neuronal subtype specification and transcriptional heterogeneity remain unclear. By comparing the gene expression profiles of single layer 6 corticothalamic neurons in somatosensory cortex, we show that transcriptional subtypes primarily reflect axonal projection pattern, laminar position within the cortex, and neuronal activity state. Pseudotemporal ordering of 1,023 cellular responses to sensory manipulation demonstrates that changes in expression of activity-induced genes both reinforced cell-type identity and contributed to increased transcriptional heterogeneity within each cell type. This is due to cell-type biased choices of transcriptional states following manipulation of neuronal activity. These results reveal that axonal projection pattern, laminar position, and activity state define significant axes of variation that contribute both to the transcriptional identity of individual neurons and to the transcriptional heterogeneity within each neuronal subtype.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S2211-1247(17)31870-3 [pii] 10.1016/j.celrep.2017.12.046 ">10.1016/j.celrep.2017.12.046 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 20  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading20">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse20" aria-expanded="true" aria-controls="collapse20">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">20. Group 1 Innate Lymphoid Cell Lineage Identity Is Determined by a cis-Regulatory Element Marked by a Long Non-coding RNA. (2017)</p>
						                 <p class="font-italic font-weight-light">Mowel WK, McCright SJ, Kotzin JJ, Collet MA, Uyar A, Chen X, DeLaney A, Spencer SP, Virtue AT, Yang E, Villarino A, Kurachi M, Dunagin MC, Pritchard GH, Stein J, Hughes C, Fonseca-Pereira D, Veiga-Fernandes H, Raj A, Kambayashi T, Brodsky IE, O'Shea JJ, Wherry EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL, Williams A, Flavell RA, Henao-Mejia J</p>
						                 <p>Immunity. 2017 Sep 19;47(3):435-449.e8. doi: 10.1016/j.immuni.2017.08.012.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse20" class="collapse" aria-labelledby="heading20" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="28930659" class="altmetric-embed pull-left"> </div>
						          <p>Commitment to the innate lymphoid cell (ILC) lineage is determined by Id2, a transcriptional regulator that antagonizes T and B cell-specific gene expression programs. Yet how Id2 expression is regulated in each ILC subset remains poorly understood. We identified a cis-regulatory element demarcated by a long non-coding RNA (lncRNA) that controls the function and lineage identity of group 1 ILCs, while being dispensable for early ILC development and homeostasis of ILC2s and ILC3s. The locus encoding this lncRNA, which we termed Rroid, directly interacted with the promoter of its neighboring gene, Id2, in group 1 ILCs. Moreover, the Rroid locus, but not the lncRNA itself, controlled the identity and function of ILC1s by promoting chromatin accessibility and deposition of STAT5 at the promoter of Id2 in response to interleukin (IL)-15. Thus, non-coding elements responsive to extracellular cues unique to each ILC subset represent a key regulatory layer for controlling the identity and function of ILCs.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S1074-7613(17)30370-9 [pii] 10.1016/j.immuni.2017.08.012 ">10.1016/j.immuni.2017.08.012 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 21  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading21">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse21" aria-expanded="true" aria-controls="collapse21">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">21. Changes in the Excitability of Neocortical Neurons in a Mouse Model of Amyotrophic Lateral Sclerosis Are Not Specific to Corticospinal Neurons and Are Modulated by Advancing Disease. (2017)</p>
						                 <p class="font-italic font-weight-light">Kim J, Hughes EG, Shetty AS, Arlotta P, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Bergles DE, Brown SP</p>
						                 <p>J Neurosci. 2017 Sep 13;37(37):9037-9053. doi: 10.1523/JNEUROSCI.0811-17.2017. Epub 2017 Aug 17.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse21" class="collapse" aria-labelledby="heading21" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="28821643" class="altmetric-embed pull-left"> </div>
						          <p>Cell type-specific changes in neuronal excitability have been proposed to contribute to the selective degeneration of corticospinal neurons in amyotrophic lateral sclerosis (ALS) and to neocortical hyperexcitability, a prominent feature of both inherited and sporadic variants of the disease, but the mechanisms underlying selective loss of specific cell types in ALS are not known. We analyzed the physiological properties of distinct classes of cortical neurons in the motor cortex of hSOD1(G93A) mice of both sexes and found that they all exhibit increases in intrinsic excitability that depend on disease stage. Targeted recordings and in vivo calcium imaging further revealed that neurons adapt their functional properties to normalize cortical excitability as the disease progresses. Although different neuron classes all exhibited increases in intrinsic excitability, transcriptional profiling indicated that the molecular mechanisms underlying these changes are cell type specific. The increases in excitability in both excitatory and inhibitory cortical neurons show that selective dysfunction of neuronal cell types cannot account for the specific vulnerability of corticospinal motor neurons in ALS. Furthermore, the stage-dependent alterations in neuronal function highlight the ability of cortical circuits to adapt as disease progresses. These findings show that both disease stage and cell type must be considered when developing therapeutic strategies for treating ALS.SIGNIFICANCE STATEMENT It is not known why certain classes of neurons preferentially die in different neurodegenerative diseases. It has been proposed that the enhanced excitability of affected neurons is a major contributor to their selective loss. We show using a mouse model of amyotrophic lateral sclerosis (ALS), a disease in which corticospinal neurons exhibit selective vulnerability, that changes in excitability are not restricted to this neuronal class and that excitability does not increase monotonically with disease progression. Moreover, although all neuronal cell types tested exhibited abnormal functional properties, analysis of their gene expression demonstrated cell type-specific responses to the ALS-causing mutation. These findings suggest that therapies for ALS may need to be tailored for different cell types and stages of disease.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1523/JNEUROSCI.0811-17.2017 ">10.1523/JNEUROSCI.0811-17.2017 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 22  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading22">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse22" aria-expanded="true" aria-controls="collapse22">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">22. A ketogenic diet rescues hippocampal memory defects in a mouse model of Kabuki syndrome. (2017)</p>
						                 <p class="font-italic font-weight-light">Benjamin JS, Pilarowski GO, Carosso GA, Zhang L, Huso DL, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Vernon HJ, Hansen KD, Bjornsson HT</p>
						                 <p>Proc Natl Acad Sci U S A. 2017 Jan 3;114(1):125-130. doi: 10.1073/pnas.1611431114. Epub 2016 Dec 20.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse22" class="collapse" aria-labelledby="heading22" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27999180" class="altmetric-embed pull-left"> </div>
						          <p>Kabuki syndrome is a Mendelian intellectual disability syndrome caused by mutations in either of two genes (KMT2D and KDM6A) involved in chromatin accessibility. We previously showed that an agent that promotes chromatin opening, the histone deacetylase inhibitor (HDACi) AR-42, ameliorates the deficiency of adult neurogenesis in the granule cell layer of the dentate gyrus and rescues hippocampal memory defects in a mouse model of Kabuki syndrome (Kmt2d(+/betaGeo)). Unlike a drug, a dietary intervention could be quickly transitioned to the clinic. Therefore, we have explored whether treatment with a ketogenic diet could lead to a similar rescue through increased amounts of beta-hydroxybutyrate, an endogenous HDACi. Here, we report that a ketogenic diet in Kmt2d(+/betaGeo) mice modulates H3ac and H3K4me3 in the granule cell layer, with concomitant rescue of both the neurogenesis defect and hippocampal memory abnormalities seen in Kmt2d(+/betaGeo) mice; similar effects on neurogenesis were observed on exogenous administration of beta-hydroxybutyrate. These data suggest that dietary modulation of epigenetic modifications through elevation of beta-hydroxybutyrate may provide a feasible strategy to treat the intellectual disability seen in Kabuki syndrome and related disorders.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1611431114 ">10.1073/pnas.1611431114 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 23  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading23">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse23" aria-expanded="true" aria-controls="collapse23">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">23. The DPYSL2 gene connects mTOR and schizophrenia. (2016)</p>
						                 <p class="font-italic font-weight-light">Pham X, Song G, Lao S, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Zhu H, Valle D, Avramopoulos D</p>
						                 <p>Transl Psychiatry. 2016 Nov 1;6(11):e933. doi: 10.1038/tp.2016.204.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse23" class="collapse" aria-labelledby="heading23" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27801893" class="altmetric-embed pull-left"> </div>
						          <p>We previously reported a schizophrenia-associated polymorphic CT di-nucleotide repeat (DNR) at the 5'-untranslated repeat (UTR) of DPYSL2, which responds to mammalian target of Rapamycin (mTOR) signaling with allelic differences in reporter assays. Now using microarray analysis, we show that the DNR alleles interact differentially with specific proteins, including the mTOR-related protein HuD/ELAVL4. We confirm the differential binding to HuD and other known mTOR effectors by electrophoretic mobility shift assays. We edit HEK293 cells by CRISPR/Cas9 to carry the schizophrenia risk variant (13DNR) and observe a significant reduction of the corresponding CRMP2 isoform. These edited cells confirm the response to mTOR inhibitors and show a twofold shortening of the cellular projections. Transcriptome analysis of these modified cells by RNA-seq shows changes in 12.7% of expressed transcripts at a false discovery rate of 0.05. These transcripts are enriched in immunity-related genes, overlap significantly with those modified by the schizophrenia-associated gene, ZNF804A, and have a reverse expression signature from that seen with antipsychotic drugs. Our results support the functional importance of the DPYSL2 DNR and a role for mTOR signaling in schizophrenia.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/tp.2016.204 ">10.1038/tp.2016.204 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 24  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading24">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse24" aria-expanded="true" aria-controls="collapse24">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">24. The long non-coding RNA Morrbid regulates Bim and short-lived myeloid cell lifespan. (2016)</p>
						                 <p class="font-italic font-weight-light">Kotzin JJ, Spencer SP, McCright SJ, Kumar DBU, Collet MA, Mowel WK, Elliott EN, Uyar A, Makiya MA, Dunagin MC, Harman CCD, Virtue AT, Zhu S, Bailis W, Stein J, Hughes C, Raj A, Wherry EJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Klion AD, Rinn JL, Williams A, Flavell RA, Henao-Mejia J</p>
						                 <p>Nature. 2016 Sep 8;537(7619):239-243. doi: 10.1038/nature19346. Epub 2016 Aug 15.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse24" class="collapse" aria-labelledby="heading24" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27525555" class="altmetric-embed pull-left"> </div>
						          <p>Neutrophils, eosinophils and 'classical' monocytes collectively account for about 70% of human blood leukocytes and are among the shortest-lived cells in the body. Precise regulation of the lifespan of these myeloid cells is critical to maintain protective immune responses and minimize the deleterious consequences of prolonged inflammation. However, how the lifespan of these cells is strictly controlled remains largely unknown. Here we identify a long non-coding RNA that we termed Morrbid, which tightly controls the survival of neutrophils, eosinophils and classical monocytes in response to pro-survival cytokines in mice. To control the lifespan of these cells, Morrbid regulates the transcription of the neighbouring pro-apoptotic gene, Bcl2l11 (also known as Bim), by promoting the enrichment of the PRC2 complex at the Bcl2l11 promoter to maintain this gene in a poised state. Notably, Morrbid regulates this process in cis, enabling allele-specific control of Bcl2l11 transcription. Thus, in these highly inflammatory cells, changes in Morrbid levels provide a locus-specific regulatory mechanism that allows rapid control of apoptosis in response to extracellular pro-survival signals. As MORRBID is present in humans and dysregulated in individuals with hypereosinophilic syndrome, this long non-coding RNA may represent a potential therapeutic target for inflammatory disorders characterized by aberrant short-lived myeloid cell lifespan.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nature19346 ">10.1038/nature19346 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 25  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading25">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse25" aria-expanded="true" aria-controls="collapse25">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">25. Investigating long noncoding RNAs using animal models. (2016)</p>
						                 <p class="font-italic font-weight-light">Feyder M, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>,</p>
						                 <p>J Clin Invest. 2016 Aug 1;126(8):2783-91. doi: 10.1172/JCI84422. Epub 2016 Aug 1.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse25" class="collapse" aria-labelledby="heading25" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27479747" class="altmetric-embed pull-left"> </div>
						          <p>The number of long noncoding RNAs (lncRNAs) has grown rapidly; however, our understanding of their function remains limited. Although cultured cells have facilitated investigations of lncRNA function at the molecular level, the use of animal models provides a rich context in which to investigate the phenotypic impact of these molecules. Promising initial studies using animal models demonstrated that lncRNAs influence a diverse number of phenotypes, ranging from subtle dysmorphia to viability. Here, we highlight the diversity of animal models and their unique advantages, discuss the use of animal models to profile lncRNA expression, evaluate experimental strategies to manipulate lncRNA function in vivo, and review the phenotypes attributable to lncRNAs. Despite a limited number of studies leveraging animal models, lncRNAs are already recognized as a notable class of molecules with important implications for health and disease.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1172/JCI84422 ">10.1172/JCI84422 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 26  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading26">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse26" aria-expanded="true" aria-controls="collapse26">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">26. Long noncoding RNAs: Central to nervous system development. (2016)</p>
						                 <p class="font-italic font-weight-light">Hart RP, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>,</p>
						                 <p>Int J Dev Neurosci. 2016 Dec;55:109-116. doi: 10.1016/j.ijdevneu.2016.06.001. Epub 2016 Jun 11.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse26" class="collapse" aria-labelledby="heading26" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="27296516" class="altmetric-embed pull-left"> </div>
						          <p>The development of the central nervous system (CNS) is a complex orchestration of stem cells, transcription factors, growth/differentiation factors, and epigenetic control. Noncoding RNAs have been identified, classified, and studied for their functional roles in many systems including the CNS. In particular, the class of long noncoding RNAs (lncRNAs) has generated both enthusiasm and skepticism due to the unexpected discovery, the diversity of mechanisms, and the lower level of expression than found in protein-coding RNAs. Here we describe evidence supporting the role of lncRNAs in driving CNS-specific differentiation. It is clear that lncRNAs exhibit a functional diversity that makes their study and compartmentalization more challenging than other classes of noncoding RNAs. We predict, however, that lncRNAs will be essential for the characterization of discrete neuronal cell types in the age of single-cell transcriptomics and that these regulatory RNAs contribute to the multitude of functional mechanisms during CNS differentiation that will rival the diversities of protein-based mechanisms.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/S0736-5748(16)30109-5 [pii] 10.1016/j.ijdevneu.2016.06.001 ">10.1016/j.ijdevneu.2016.06.001 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 27  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading27">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse27" aria-expanded="true" aria-controls="collapse27">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">27. Creation and characterization of an airway epithelial cell line for stable expression of CFTR variants. (2016)</p>
						                 <p class="font-italic font-weight-light">Gottschalk LB, Vecchio-Pagan B, Sharma N, Han ST, Franca A, Wohler ES, Batista DA, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Cutting GR</p>
						                 <p>J Cyst Fibros. 2016 May;15(3):285-94. doi: 10.1016/j.jcf.2015.11.010. Epub 2015 Dec 13.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse27" class="collapse" aria-labelledby="heading27" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="26694805" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: Analysis of the functional consequences and treatment response of rare CFTR variants is challenging due to the limited availability of primary airways cells. METHODS: A Flp recombination target (FRT) site for stable expression of CFTR was incorporated into an immortalized CF bronchial epithelial cell line (CFBE41o-). CFTR cDNA was integrated into the FRT site. Expression was evaluated by western blotting and confocal microscopy and function measured by short circuit current. RNA sequencing was used to compare the transcriptional profile of the resulting CF8Flp cell line to primary cells and tissues. RESULTS: Functional CFTR was expressed from integrated cDNA at the FRT site of the CF8Flp cell line at levels comparable to that seen in native airway cells. CF8Flp cells expressing WT-CFTR have a stable transcriptome comparable to that of primary cultured airway epithelial cells, including genes that play key roles in CFTR pathways. CONCLUSION: CF8Flp cells provide a viable substitute for primary CF airway cells for the analysis of CFTR variants in a native context.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.jcf.2015.11.010 ">10.1016/j.jcf.2015.11.010 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 28  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading28">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse28" aria-expanded="true" aria-controls="collapse28">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">28. Linking RNA biology to lncRNAs. (2015)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL</p>
						                 <p>Genome Res. 2015 Oct;25(10):1456-65. doi: 10.1101/gr.191122.115.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse28" class="collapse" aria-labelledby="heading28" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="26430155" class="altmetric-embed pull-left"> </div>
						          <p>The regulatory potential of RNA has never ceased to amaze: from RNA catalysis, to RNA-mediated splicing, to RNA-based silencing of an entire chromosome during dosage compensation. More recently, thousands of long noncoding RNA (lncRNA) transcripts have been identified, the majority with unknown function. Thus, it is tempting to think that these lncRNAs represent a cadre of new factors that function through ribonucleic mechanisms. Some evidence points to several lncRNAs with tantalizing physiological contributions and thought-provoking molecular modalities. However, dissecting the RNA biology of lncRNAs has been difficult, and distinguishing the independent contributions of functional RNAs from underlying DNA elements, or the local act of transcription, is challenging. Here, we aim to survey the existing literature and highlight future approaches that will be needed to link the RNA-based biology and mechanisms of lncRNAs in vitro and in vivo.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1101/gr.191122.115 ">10.1101/gr.191122.115 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 29  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading29">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse29" aria-expanded="true" aria-controls="collapse29">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">29. Spatiotemporal expression and transcriptional perturbations by long noncoding RNAs in the mouse brain. (2015)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Groff AF, Sauvageau M, Trayes-Gibson Z, Sanchez-Gomez DB, Morse M, Martin RD, Elcavage LE, Liapis SC, Gonzalez-Celeiro M, Plana O, Li E, Gerhardinger C, Tomassy GS, Arlotta P, Rinn JL</p>
						                 <p>Proc Natl Acad Sci U S A. 2015 Jun 2;112(22):6855-62. doi: 10.1073/pnas.1411263112.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse29" class="collapse" aria-labelledby="heading29" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="26034286" class="altmetric-embed pull-left"> </div>
						          <p>Long noncoding RNAs (lncRNAs) have been implicated in numerous cellular processes including brain development. However, the in vivo expression dynamics and molecular pathways regulated by these loci are not well understood. Here, we leveraged a cohort of 13 lncRNAnull mutant mouse models to investigate the spatiotemporal expression of lncRNAs in the developing and adult brain and the transcriptome alterations resulting from the loss of these lncRNA loci. We show that several lncRNAs are differentially expressed both in time and space, with some presenting highly restricted expression in only selected brain regions. We further demonstrate altered regulation of genes for a large variety of cellular pathways and processes upon deletion of the lncRNA loci. Finally, we found that 4 of the 13 lncRNAs significantly affect the expression of several neighboring proteincoding genes in a cis-like manner. By providing insight into the endogenous expression patterns and the transcriptional perturbations caused by deletion of the lncRNA locus in the developing and postnatal mammalian brain, these data provide a resource to facilitate future examination of the specific functional relevance of these genes in neural development, brain function, and disease.</p>
						          
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26034286">Article</a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 30  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading30">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse30" aria-expanded="true" aria-controls="collapse30">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">30. DeCoN: genome-wide analysis of in vivo transcriptional dynamics during pyramidal neuron fate selection in neocortex. (2015)</p>
						                 <p class="font-italic font-weight-light">Molyneaux BJ, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Brettler AC, Chen HH, Hrvatin S, Rinn JL, Arlotta P</p>
						                 <p>Neuron. 2015 Jan 21;85(2):275-288. doi: 10.1016/j.neuron.2014.12.024. Epub 2014 Dec 31.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse30" class="collapse" aria-labelledby="heading30" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="25556833" class="altmetric-embed pull-left"> </div>
						          <p>UNLABELLED: Neuronal development requires a complex choreography of transcriptional decisions to obtain specific cellular identities. Realizing the ultimate goal of identifying genome-wide signatures that define and drive specific neuronal fates has been hampered by enormous complexity in both time and space during development. Here, we have paired high-throughput purification of pyramidal neuron subclasses with deep profiling of spatiotemporal transcriptional dynamics during corticogenesis to resolve lineage choice decisions. We identified numerous features ranging from spatial and temporal usage of alternative mRNA isoforms and promoters to a host of mRNA genes modulated during fate specification. Notably, we uncovered numerous long noncoding RNAs with restricted temporal and cell-type-specific expression. To facilitate future exploration, we provide an interactive online database to enable multidimensional data mining and dissemination. This multifaceted study generates a powerful resource and informs understanding of the transcriptional regulation underlying pyramidal neuron diversity in the neocortex. VIDEO ABSTRACT:</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.neuron.2014.12.024 ">10.1016/j.neuron.2014.12.024 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 31  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading31">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse31" aria-expanded="true" aria-controls="collapse31">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">31. Gene co-regulation by Fezf2 selects neurotransmitter identity and connectivity of corticospinal neurons. (2014)</p>
						                 <p class="font-italic font-weight-light">Lodato S, Molyneaux BJ, Zuccaro E, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Chen HH, Yuan W, Meleski A, Takahashi E, Mahony S, Rinn JL, Gifford DK, Arlotta P</p>
						                 <p>Nat Neurosci. 2014 Aug;17(8):1046-54. doi: 10.1038/nn.3757. Epub 2014 Jul 6.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse31" class="collapse" aria-labelledby="heading31" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24997765" class="altmetric-embed pull-left"> </div>
						          <p>The neocortex contains an unparalleled diversity of neuronal subtypes, each defined by distinct traits that are developmentally acquired under the control of subtype-specific and pan-neuronal genes. The regulatory logic that orchestrates the expression of these unique combinations of genes is unknown for any class of cortical neuron. Here, we report that Fezf2 is a selector gene able to regulate the expression of gene sets that collectively define mouse corticospinal motor neurons (CSMN). We find that Fezf2 directly induces the glutamatergic identity of CSMN via activation of Vglut1 (Slc17a7) and inhibits a GABAergic fate by repressing transcription of Gad1. In addition, we identify the axon guidance receptor EphB1 as a target of Fezf2 necessary to execute the ipsilateral extension of the corticospinal tract. Our data indicate that co-regulated expression of neuron subtype-specific and pan-neuronal gene batteries by a single transcription factor is one component of the regulatory logic responsible for the establishment of CSMN identity.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nn.3757 ">10.1038/nn.3757 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 32  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading32">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse32" aria-expanded="true" aria-controls="collapse32">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">32. A positive feedback mechanism that regulates expression of miR-9 during neurogenesis. (2014)</p>
						                 <p class="font-italic font-weight-light">Davila JL, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Ricupero CL, Camarillo C, Oni EN, Swerdel MR, Toro-Ramos AJ, Li J, Hart RP</p>
						                 <p>PLoS One. 2014 Apr 8;9(4):e94348. doi: 10.1371/journal.pone.0094348. eCollection 2014.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse32" class="collapse" aria-labelledby="heading32" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24714615" class="altmetric-embed pull-left"> </div>
						          <p>MiR-9, a neuron-specific miRNA, is an important regulator of neurogenesis. In this study we identify how miR-9 is regulated during early differentiation from a neural stem-like cell. We utilized two immortalized rat precursor clones, one committed to neurogenesis (L2.2) and another capable of producing both neurons and non-neuronal cells (L2.3), to reproducibly study early neurogenesis. Exogenous miR-9 is capable of increasing neurogenesis from L2.3 cells. Only one of three genomic loci capable of encoding miR-9 was regulated during neurogenesis and the promoter region of this locus contains sufficient functional elements to drive expression of a luciferase reporter in a developmentally regulated pattern. Furthermore, among a large number of potential regulatory sites encoded in this sequence, Mef2 stood out because of its known pro-neuronal role. Of four Mef2 paralogs, we found only Mef2C mRNA was regulated during neurogenesis. Removal of predicted Mef2 binding sites or knockdown of Mef2C expression reduced miR-9-2 promoter activity. Finally, the mRNA encoding the Mef2C binding partner HDAC4 was shown to be targeted by miR-9. Since HDAC4 protein could be co-immunoprecipitated with Mef2C protein or with genomic Mef2 binding sequences, we conclude that miR-9 regulation is mediated, at least in part, by Mef2C binding but that expressed miR-9 has the capacity to reduce inhibitory HDAC4, stabilizing its own expression in a positive feedback mechanism.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0094348 ">10.1371/journal.pone.0094348 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 33  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading33">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse33" aria-expanded="true" aria-controls="collapse33">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">33. Topological organization of multichromosomal regions by the long intergenic noncoding RNA Firre. (2014)</p>
						                 <p class="font-italic font-weight-light">Hacisuleyman E, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Trapnell C, Williams A, Henao-Mejia J, Sun L, McClanahan P, Hendrickson DG, Sauvageau M, Kelley DR, Morse M, Engreitz J, Lander ES, Guttman M, Lodish HF, Flavell R, Raj A, Rinn JL</p>
						                 <p>Nat Struct Mol Biol. 2014 Feb;21(2):198-206. doi: 10.1038/nsmb.2764. Epub 2014 Jan 26.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse33" class="collapse" aria-labelledby="heading33" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24463464" class="altmetric-embed pull-left"> </div>
						          <p>RNA, including long noncoding RNA (lncRNA), is known to be an abundant and important structural component of the nuclear matrix. However, the molecular identities, functional roles and localization dynamics of lncRNAs that influence nuclear architecture remain poorly understood. Here, we describe one lncRNA, Firre, that interacts with the nuclear-matrix factor hnRNPU through a 156-bp repeating sequence and localizes across an ~5-Mb domain on the X chromosome. We further observed Firre localization across five distinct trans-chromosomal loci, which reside in spatial proximity to the Firre genomic locus on the X chromosome. Both genetic deletion of the Firre locus and knockdown of hnRNPU resulted in loss of colocalization of these trans-chromosomal interacting loci. Thus, our data suggest a model in which lncRNAs such as Firre can interface with and modulate nuclear architecture across chromosomes.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2764 ">10.1038/nsmb.2764 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 34  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading34">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse34" aria-expanded="true" aria-controls="collapse34">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">34. RNase-mediated protein footprint sequencing reveals protein-binding sites throughout the human transcriptome. (2014)</p>
						                 <p class="font-italic font-weight-light">Silverman IM, Li F, Alexander A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Trapnell C, Rinn JL, Gregory BD</p>
						                 <p>Genome Biol. 2014 Jan 7;15(1):R3. doi: 10.1186/gb-2014-15-1-r3.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse34" class="collapse" aria-labelledby="heading34" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24393486" class="altmetric-embed pull-left"> </div>
						          <p>Although numerous approaches have been developed to map RNA-binding sites of individual RNA-binding proteins (RBPs), few methods exist that allow assessment of global RBP-RNA interactions. Here, we describe PIP-seq, a universal, high-throughput, ribonuclease-mediated protein footprint sequencing approach that reveals RNA-protein interaction sites throughout a transcriptome of interest. We apply PIP-seq to the HeLa transcriptome and compare binding sites found using different cross-linkers and ribonucleases. From this analysis, we identify numerous putative RBP-binding motifs, reveal novel insights into co-binding by RBPs, and uncover a significant enrichment for disease-associated polymorphisms within RBP interaction sites.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1186/gb-2014-15-1-r3 ">10.1186/gb-2014-15-1-r3 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 35  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading35">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse35" aria-expanded="true" aria-controls="collapse35">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">35. Multiple knockout mouse models reveal lincRNAs are required for life and brain development. (2013)</p>
						                 <p class="font-italic font-weight-light">Sauvageau M, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Lodato S, Bonev B, Groff AF, Gerhardinger C, Sanchez-Gomez DB, Hacisuleyman E, Li E, Spence M, Liapis SC, Mallard W, Morse M, Swerdel MR, D'Ecclessis MF, Moore JC, Lai V, Gong G, Yancopoulos GD, Frendewey D, Kellis M, Hart RP, Valenzuela DM, Arlotta P, Rinn JL</p>
						                 <p>Elife. 2013 Dec 31;2:e01749. doi: 10.7554/eLife.01749.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse35" class="collapse" aria-labelledby="heading35" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24381249" class="altmetric-embed pull-left"> </div>
						          <p>Many studies are uncovering functional roles for long noncoding RNAs (lncRNAs), yet few have been tested for in vivo relevance through genetic ablation in animal models. To investigate the functional relevance of lncRNAs in various physiological conditions, we have developed a collection of 18 lncRNA knockout strains in which the locus is maintained transcriptionally active. Initial characterization revealed peri- and postnatal lethal phenotypes in three mutant strains (Fendrr, Peril, and Mdgt), the latter two exhibiting incomplete penetrance and growth defects in survivors. We also report growth defects for two additional mutant strains (linc-Brn1b and linc-Pint). Further analysis revealed defects in lung, gastrointestinal tract, and heart in Fendrr(-/-) neonates, whereas linc-Brn1b(-/-) mutants displayed distinct abnormalities in the generation of upper layer II-IV neurons in the neocortex. This study demonstrates that lncRNAs play critical roles in vivo and provides a framework and impetus for future larger-scale functional investigation into the roles of lncRNA molecules. DOI: http://dx.doi.org/10.7554/eLife.01749.001.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.7554/eLife.01749 ">10.7554/eLife.01749 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 36  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading36">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse36" aria-expanded="true" aria-controls="collapse36">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">36. Poly-combing the genome for RNA. (2013)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL</p>
						                 <p>Nat Struct Mol Biol. 2013 Dec;20(12):1344-6. doi: 10.1038/nsmb.2728.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse36" class="collapse" aria-labelledby="heading36" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24304912" class="altmetric-embed pull-left"> </div>
						          <p>An unresolved question in mammalian epigenetic regulation is how ubiquitously expressed chromatin-modifying complexes such as Polycomb group complex 2 (PRC2) find their specific target sites across an intricate choreography of localization events in time and space. Two recent studies now provide critical new insights into an intriguing genome-wide role for RNA in PRC2 regulation.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nsmb.2728 ">10.1038/nsmb.2728 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 37  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading37">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse37" aria-expanded="true" aria-controls="collapse37">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">37. DNMT1-interacting RNAs block gene-specific DNA methylation. (2013)</p>
						                 <p class="font-italic font-weight-light">Di Ruscio A, Ebralidze AK, Benoukraf T, Amabile G, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Terragni J, Figueroa ME, De Figueiredo Pontes LL, Alberich-Jorda M, Zhang P, Wu M, D'Alo F, Melnick A, Leone G, Ebralidze KK, Pradhan S, Rinn JL, Tenen DG</p>
						                 <p>Nature. 2013 Nov 21;503(7476):371-6. doi: 10.1038/nature12598. Epub 2013 Oct 9.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse37" class="collapse" aria-labelledby="heading37" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="24107992" class="altmetric-embed pull-left"> </div>
						          <p>DNA methylation was first described almost a century ago; however, the rules governing its establishment and maintenance remain elusive. Here we present data demonstrating that active transcription regulates levels of genomic methylation. We identify a novel RNA arising from the CEBPA gene locus that is critical in regulating the local DNA methylation profile. This RNA binds to DNMT1 and prevents CEBPA gene locus methylation. Deep sequencing of transcripts associated with DNMT1 combined with genome-scale methylation and expression profiling extend the generality of this finding to numerous gene loci. Collectively, these results delineate the nature of DNMT1-RNA interactions and suggest strategies for gene-selective demethylation of therapeutic targets in human diseases.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nature12598 ">10.1038/nature12598 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 38  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading38">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse38" aria-expanded="true" aria-controls="collapse38">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">38. The microRNA miR-181 is a critical cellular metabolic rheostat essential for NKT cell ontogenesis and lymphocyte development and homeostasis. (2013)</p>
						                 <p class="font-italic font-weight-light">Henao-Mejia J, Williams A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Staron M, Licona-Limon P, Kaech SM, Nakayama M, Rinn JL, Flavell RA</p>
						                 <p>Immunity. 2013 May 23;38(5):984-97. doi: 10.1016/j.immuni.2013.02.021. Epub 2013 Apr 25.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse38" class="collapse" aria-labelledby="heading38" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="23623381" class="altmetric-embed pull-left"> </div>
						          <p>Regulation of metabolic pathways in the immune system provides a mechanism to actively control cellular function, growth, proliferation, and survival. Here, we report that miR-181 is a nonredundant determinant of cellular metabolism and is essential for supporting the biosynthetic demands of early NKT cell development. As a result, miR-181-deficient mice showed a complete absence of mature NKT cells in the thymus and periphery. Mechanistically, miR-181 modulated expression of the phosphatase PTEN to control PI3K signaling, which was a primary stimulus for anabolic metabolism in immune cells. Thus miR-181-deficient mice also showed severe defects in lymphoid development and T cell homeostasis associated with impaired PI3K signaling. These results uncover miR-181 as essential for NKT cell development and establish this family of miRNAs as central regulators of PI3K signaling and global metabolic fitness during development and homeostasis.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.immuni.2013.02.021 ">10.1016/j.immuni.2013.02.021 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 39  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading39">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse39" aria-expanded="true" aria-controls="collapse39">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">39. Long noncoding RNAs regulate adipogenesis. (2013)</p>
						                 <p class="font-italic font-weight-light">Sun L, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Trapnell C, Alexander R, Lo KA, Hacisuleyman E, Sauvageau M, Tazon-Vega B, Kelley DR, Hendrickson DG, Yuan B, Kellis M, Lodish HF, Rinn JL</p>
						                 <p>Proc Natl Acad Sci U S A. 2013 Feb 26;110(9):3387-92. doi: 10.1073/pnas.1222643110. Epub 2013 Feb 11.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse39" class="collapse" aria-labelledby="heading39" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="23401553" class="altmetric-embed pull-left"> </div>
						          <p>The prevalence of obesity has led to a surge of interest in understanding the detailed mechanisms underlying adipocyte development. Many protein-coding genes, mRNAs, and microRNAs have been implicated in adipocyte development, but the global expression patterns and functional contributions of long noncoding RNA (lncRNA) during adipogenesis have not been explored. Here we profiled the transcriptome of primary brown and white adipocytes, preadipocytes, and cultured adipocytes and identified 175 lncRNAs that are specifically regulated during adipogenesis. Many lncRNAs are adipose-enriched, strongly induced during adipogenesis, and bound at their promoters by key transcription factors such as peroxisome proliferator-activated receptor gamma (PPARgamma) and CCAAT/enhancer-binding protein alpha (CEBPalpha). RNAi-mediated loss of function screens identified functional lncRNAs with varying impact on adipogenesis. Collectively, we have identified numerous lncRNAs that are functionally required for proper adipogenesis.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1073/pnas.1222643110 ">10.1073/pnas.1222643110 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 40  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading40">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse40" aria-expanded="true" aria-controls="collapse40">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">40. Differential analysis of gene regulation at transcript resolution with RNA-seq. (2013)</p>
						                 <p class="font-italic font-weight-light">Trapnell C, Hendrickson DG, Sauvageau M, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL, Pachter L</p>
						                 <p>Nat Biotechnol. 2013 Jan;31(1):46-53. doi: 10.1038/nbt.2450. Epub 2012 Dec 9.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse40" class="collapse" aria-labelledby="heading40" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="23222703" class="altmetric-embed pull-left"> </div>
						          <p>Differential analysis of gene and transcript expression using high-throughput RNA sequencing (RNA-seq) is complicated by several sources of measurement variability and poses numerous statistical challenges. We present Cuffdiff 2, an algorithm that estimates expression at transcript-level resolution and controls for variability evident across replicate libraries. Cuffdiff 2 robustly identifies differentially expressed transcripts and genes and reveals differential splicing and promoter-preference changes. We demonstrate the accuracy of our approach through differential analysis of lung fibroblasts in response to loss of the developmental transcription factor HOXA1, which we show is required for lung fibroblast and HeLa cell cycle progression. Loss of HOXA1 results in significant expression level changes in thousands of individual transcripts, along with isoform switching events in key regulators of the cell cycle. Cuffdiff 2 performs robust differential analysis in RNA-seq experiments at transcript resolution, revealing a layer of regulation not readily observable with other high-throughput technologies.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nbt.2450 ">10.1038/nbt.2450 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 41  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading41">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse41" aria-expanded="true" aria-controls="collapse41">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">41. Computational analysis of noncoding RNAs. (2012)</p>
						                 <p class="font-italic font-weight-light">Washietl S, Will S, Hendrix DA, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Rinn JL, Berger B, Kellis M</p>
						                 <p>Wiley Interdiscip Rev RNA. 2012 Nov-Dec;3(6):759-78. doi: 10.1002/wrna.1134. Epub 2012 Sep 18.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse41" class="collapse" aria-labelledby="heading41" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="22991327" class="altmetric-embed pull-left"> </div>
						          <p>Noncoding RNAs have emerged as important key players in the cell. Understanding their surprisingly diverse range of functions is challenging for experimental and computational biology. Here, we review computational methods to analyze noncoding RNAs. The topics covered include basic and advanced techniques to predict RNA structures, annotation of noncoding RNAs in genomic data, mining RNA-seq data for novel transcripts and prediction of transcript structures, computational aspects of microRNAs, and database resources.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1002/wrna.1134 ">10.1002/wrna.1134 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 42  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading42">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse42" aria-expanded="true" aria-controls="collapse42">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">42. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. (2012)</p>
						                 <p class="font-italic font-weight-light">Trapnell C, Roberts A, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Pertea G, Kim D, Kelley DR, Pimentel H, Salzberg SL, Rinn JL, Pachter L</p>
						                 <p>Nat Protoc. 2012 Mar 1;7(3):562-78. doi: 10.1038/nprot.2012.016.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse42" class="collapse" aria-labelledby="heading42" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="22383036" class="altmetric-embed pull-left"> </div>
						          <p>Recent advances in high-throughput cDNA sequencing (RNA-seq) can reveal new genes and splice variants and quantify expression genome-wide in a single assay. The volume and complexity of data from RNA-seq experiments necessitate scalable, fast and mathematically principled analysis software. TopHat and Cufflinks are free, open-source software tools for gene discovery and comprehensive expression analysis of high-throughput mRNA sequencing (RNA-seq) data. Together, they allow biologists to identify new genes and new splice variants of known ones, as well as compare gene and transcript expression under two or more conditions. This protocol describes in detail how to use TopHat and Cufflinks to perform such analyses. It also covers several accessory tools and utilities that aid in managing data, including CummeRbund, a tool for visualizing RNA-seq analysis results. Although the procedure assumes basic informatics skills, these tools assume little to no background with RNA-seq analysis and are meant for novices and experts alike. The protocol begins with raw sequencing reads and produces a transcriptome assembly, lists of differentially expressed and regulated genes and transcripts, and publication-quality visualizations of analysis results. The protocol's execution time depends on the volume of transcriptome sequencing data and available computing resources but takes less than 1 d of computer time for typical experiments and approximately 1 h of hands-on time.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1038/nprot.2012.016 ">10.1038/nprot.2012.016 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 43  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading43">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse43" aria-expanded="true" aria-controls="collapse43">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">43. Expression profiling of synaptic microRNAs from the adult rat brain identifies regional differences and seizure-induced dynamic modulation. (2012)</p>
						                 <p class="font-italic font-weight-light">Pichardo-Casas I, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Swerdel MR, Athie A, Davila J, Ramos-Brossier M, Lapid-Volosin M, Friedman WJ, Hart RP, Vaca L</p>
						                 <p>Brain Res. 2012 Feb 3;1436:20-33. doi: 10.1016/j.brainres.2011.12.001. Epub 2011 Dec 9.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse43" class="collapse" aria-labelledby="heading43" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="22197703" class="altmetric-embed pull-left"> </div>
						          <p>In recent years, microRNAs or miRNAs have been proposed to target neuronal mRNAs localized near the synapse, exerting a pivotal role in modulating local protein synthesis, and presumably affecting adaptive mechanisms such as synaptic plasticity. In the present study we have characterized the distribution of miRNAs in five regions of the adult mammalian brain and compared the relative abundance between total fractions and purified synaptoneurosomes (SN), using three different methodologies. The results show selective enrichment or depletion of some miRNAs when comparing total versus SN fractions. These miRNAs were different for each brain region explored. Changes in distribution could not be attributed to simple diffusion or to a targeting sequence inside the miRNAs. In silico analysis suggest that the differences in distribution may be related to the preferential concentration of synaptically localized mRNA targeted by the miRNAs. These results favor a model of co-transport of the miRNA-mRNA complex to the synapse, although further studies are required to validate this hypothesis. Using an in vivo model for increasing excitatory activity in the cortex and the hippocampus indicates that the distribution of some miRNAs can be modulated by enhanced neuronal (epileptogenic) activity. All these results demonstrate the dynamic modulation in the local distribution of miRNAs from the adult brain, which may play key roles in controlling localized protein synthesis at the synapse.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.brainres.2011.12.001 ">10.1016/j.brainres.2011.12.001 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 44  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading44">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse44" aria-expanded="true" aria-controls="collapse44">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">44. Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. (2011)</p>
						                 <p class="font-italic font-weight-light">Cabili MN, Trapnell C, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Koziol M, Tazon-Vega B, Regev A, Rinn JL</p>
						                 <p>Genes Dev. 2011 Sep 15;25(18):1915-27. doi: 10.1101/gad.17446611. Epub 2011 Sep 2.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse44" class="collapse" aria-labelledby="heading44" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="21890647" class="altmetric-embed pull-left"> </div>
						          <p>Large intergenic noncoding RNAs (lincRNAs) are emerging as key regulators of diverse cellular processes. Determining the function of individual lincRNAs remains a challenge. Recent advances in RNA sequencing (RNA-seq) and computational methods allow for an unprecedented analysis of such transcripts. Here, we present an integrative approach to define a reference catalog of >8000 human lincRNAs. Our catalog unifies previously existing annotation sources with transcripts we assembled from RNA-seq data collected from approximately 4 billion RNA-seq reads across 24 tissues and cell types. We characterize each lincRNA by a panorama of >30 properties, including sequence, structural, transcriptional, and orthology features. We found that lincRNA expression is strikingly tissue-specific compared with coding genes, and that lincRNAs are typically coexpressed with their neighboring genes, albeit to an extent similar to that of pairs of neighboring protein-coding genes. We distinguish an additional subset of transcripts that have high evolutionary conservation but may include short ORFs and may serve as either lincRNAs or small peptides. Our integrated, comprehensive, yet conservative reference catalog of human lincRNAs reveals the global properties of lincRNAs and will facilitate experimental studies and further functional classification of these genes.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1101/gad.17446611 ">10.1101/gad.17446611 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 45  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading45">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse45" aria-expanded="true" aria-controls="collapse45">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">45. Differential regulation of microRNA stability. (2010)</p>
						                 <p class="font-italic font-weight-light">Bail S, Swerdel M, Liu H, Jiao X, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Hart RP, Kiledjian M</p>
						                 <p>RNA. 2010 May;16(5):1032-9. doi: 10.1261/rna.1851510. Epub 2010 Mar 26.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse45" class="collapse" aria-labelledby="heading45" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="20348442" class="altmetric-embed pull-left"> </div>
						          <p>MicroRNAs (miRNAs) are endogenous single-stranded RNA molecules of about 21 nucleotides in length that are fundamental post-transcriptional regulators of gene expression. Although the transcriptional and processing events involved in the generation of miRNAs have been extensively studied, very little is known pertaining to components that regulate the stability of individual miRNAs. All RNAs have distinct inherent half-lives that dictate their level of accumulation and miRNAs would be expected to follow a similar principle. Here we demonstrate that although most miRNA appear to be stable, like mRNAs, miRNAs possess differential stability in human cells. In particular, we found that miR-382, a miRNA that contributes to HIV-1 provirus latency, is unstable in cells. To determine the region of miR-382 responsible for its rapid decay, we developed a cell-free system that recapitulated the observed cell-based-regulated miR-382 turnover. The system utilizes in vitro-processed mature miRNA derived from pre-miRNA and follows the decay of the processed miRNA. Using this system, we demonstrate that instability of miR-382 is driven by sequences outside its seed region and required the 3' terminal seven nucleotides where mutations in this region increased the stability of the RNA. Moreover, the exosome 3'-5' exoribonuclease complex was identified as the primary nuclease involved in miR-382 decay with a more modest contribution by the Xrn1 and no detectable contribution by Xrn2. These studies provide evidence for an miRNA element essential for rapid miRNA decay and implicate the exosome in this process. The development of a biochemically amendable system to analyze the mechanism of differential miRNA stability provides an important step in efforts to regulate gene expression by modulating miRNA stability.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1261/rna.1851510 ">10.1261/rna.1851510 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 46  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading46">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse46" aria-expanded="true" aria-controls="collapse46">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">46. Ago2 immunoprecipitation identifies predicted microRNAs in human embryonic stem cells and neural precursors. (2009)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Davila J, Swerdel MR, Moore JC, Cohen RI, Wu H, Sun YE, Hart RP</p>
						                 <p>PLoS One. 2009 Sep 28;4(9):e7192. doi: 10.1371/journal.pone.0007192.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse46" class="collapse" aria-labelledby="heading46" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="19784364" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: MicroRNAs are required for maintenance of pluripotency as well as differentiation, but since more microRNAs have been computationally predicted in genome than have been found, there are likely to be undiscovered microRNAs expressed early in stem cell differentiation. METHODOLOGY/PRINCIPAL FINDINGS: SOLiD ultra-deep sequencing identified >10(7) unique small RNAs from human embryonic stem cells (hESC) and neural-restricted precursors that were fit to a model of microRNA biogenesis to computationally predict 818 new microRNA genes. These predicted genomic loci are associated with chromatin patterns of modified histones that are predictive of regulated gene expression. 146 of the predicted microRNAs were enriched in Ago2-containing complexes along with 609 known microRNAs, demonstrating association with a functional RISC complex. This Ago2 IP-selected subset was consistently expressed in four independent hESC lines and exhibited complex patterns of regulation over development similar to previously-known microRNAs, including pluripotency-specific expression in both hESC and iPS cells. More than 30% of the Ago2 IP-enriched predicted microRNAs are new members of existing families since they share seed sequences with known microRNAs. CONCLUSIONS/SIGNIFICANCE: Extending the classic definition of microRNAs, this large number of new microRNA genes, the majority of which are less conserved than their canonical counterparts, likely represent evolutionarily recent regulators of early differentiation. The enrichment in Ago2 containing complexes, the presence of chromatin marks indicative of regulated gene expression, and differential expression over development all support the identification of 146 new microRNAs active during early hESC differentiation.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0007192 ">10.1371/journal.pone.0007192 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 47  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading47">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse47" aria-expanded="true" aria-controls="collapse47">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">47. Rapid induction of genes associated with tissue protection and neural development in contused adult spinal cord after radial glial cell transplantation. (2009)</p>
						                 <p class="font-italic font-weight-light">Chang YW, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Li H, Kane-Goldsmith N, Tzatzalos E, Hart RP, Young W, Grumet M</p>
						                 <p>J Neurotrauma. 2009 Jul;26(7):979-93. doi: 10.1089/neu.2008.0762.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse47" class="collapse" aria-labelledby="heading47" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="19257808" class="altmetric-embed pull-left"> </div>
						          <p>Cell-based therapy has been widely evaluated in spinal cord injury (SCI) animal models and shown to improve functional recovery. However, host response to cell transplants at gene expression level is rarely discussed. We reported previously that acute transplantation of radial glial cells RG3.6 following SCI promoted early locomotion improvement within 1 week post-injury. To identify rapid molecular changes induced by RG3.6 transplantation in the host tissue, distal spinal cord segments were subjected to microarray analysis. Although RG3.6 transplantation, reduced activity of macrophages as early as 1-2 weeks post-injury, the expression levels of inflammatory genes (e.g., IL-6, MIP-2, MCP-1) were not decreased by RG3.6 treatment as compared to medium or other cell controls at 6-12 h post-injury. However, genes associated with tissue protection (Hsp70 and Hsp32) and neural cell development (Foxg1, Top2a, Sox11, Nkx2.2, Vimentin) were found to be significantly up-regulated by RG3.6 transplants. Foxg1 was the most highly induced gene in the RG3.6-treated spinal cords, and its expression by immunocytochemistry was confirmed in the host tissue. Moreover, RG3.6 treatment boosted the number of Nkx2.2 cells in the spinal cord, and these cells frequently co-expressed NG2, which marks progenitor cells. Taken together, these results demonstrate that radial glial transplants induced rapid and specific gene expression in the injured host tissue, and suggest that these early responses are associated with mechanisms of tissue protection and activation of endogenous neural progenitor cells.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1089/neu.2008-0762 ">10.1089/neu.2008-0762 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 48  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading48">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse48" aria-expanded="true" aria-controls="collapse48">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">48. Functional differentiation of a clone resembling embryonic cortical interneuron progenitors. (2008)</p>
						                 <p class="font-italic font-weight-light">Li H, Han YR, Bi C, Davila J, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Thompson K, Swerdel M, Camarillo C, Ricupero CL, Hart RP, Plummer MR, Grumet M</p>
						                 <p>Dev Neurobiol. 2008 Dec;68(14):1549-64. doi: 10.1002/dneu.20679.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse48" class="collapse" aria-labelledby="heading48" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="18814314" class="altmetric-embed pull-left"> </div>
						          <p>We have generated clones (L2.3 and RG3.6) of neural progenitors with radial glial properties from rat E14.5 cortex that differentiate into astrocytes, neurons, and oligodendrocytes. Here, we describe a different clone (L2.2) that gives rise exclusively to neurons, but not to glia. Neuronal differentiation of L2.2 cells was inhibited by bone morphogenic protein 2 (BMP2) and enhanced by Sonic Hedgehog (SHH) similar to cortical interneuron progenitors. Compared with L2.3, differentiating L2.2 cells expressed significantly higher levels of mRNAs for glutamate decarboxylases (GADs), DLX transcription factors, calretinin, calbindin, neuropeptide Y (NPY), and somatostatin. Increased levels of DLX-2, GADs, and calretinin proteins were confirmed upon differentiation. L2.2 cells differentiated into neurons that fired action potentials in vitro, and their electrophysiological differentiation was accelerated and more complete when cocultured with developing astroglial cells but not with conditioned medium from these cells. The combined results suggest that clone L2.2 resembles GABAergic interneuron progenitors in the developing forebrain.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1002/dneu.20679 ">10.1002/dneu.20679 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 49  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading49">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse49" aria-expanded="true" aria-controls="collapse49">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">49. Differentiating human multipotent mesenchymal stromal cells regulate microRNAs: prediction of microRNA regulation by PDGF during osteogenesis. (2008)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Boucher S, Ricupero CL, Fenstermacher S, Swerdel M, Chase LG, Adams CC, Chesnut J, Lakshmipathy U, Hart RP</p>
						                 <p>Exp Hematol. 2008 Oct;36(10):1354-1369. doi: 10.1016/j.exphem.2008.05.004. Epub 2008 Jul 26.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse49" class="collapse" aria-labelledby="heading49" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="18657893" class="altmetric-embed pull-left"> </div>
						          <p>OBJECTIVE: Human multipotent mesenchymal stromal cells (MSC) have the potential to differentiate into multiple cell types, although little is known about factors that control their fate. Differentiation-specific microRNAs may play a key role in stem cell self-renewal and differentiation. We propose that specific intracellular signaling pathways modulate gene expression during differentiation by regulating microRNA expression. MATERIALS AND METHODS: Illumina mRNA and NCode microRNA expression analyses were performed on MSC and their differentiated progeny. A combination of bioinformatic prediction and pathway inhibition was used to identify microRNAs associated with platelet-derived growth factor (PDGF) signaling. RESULTS: The pattern of microRNA expression in MSC is distinct from that in pluripotent stem cells, such as human embryonic stem cells. Specific populations of microRNAs are regulated in MSC during differentiation targeted toward specific cell types. Complementary mRNA expression analysis increases the pool of markers characteristic of MSC or differentiated progeny. To identify microRNA expression patterns affected by signaling pathways, we examined the PDGF pathway found to be regulated during osteogenesis by microarray studies. A set of microRNAs bioinformatically predicted to respond to PDGF signaling was experimentally confirmed by direct PDGF inhibition. CONCLUSION: Our results demonstrate that a subset of microRNAs regulated during osteogenic differentiation of MSCs is responsive to perturbation of the PDGF pathway. This approach not only identifies characteristic classes of differentiation-specific mRNAs and microRNAs, but begins to link regulated molecules with specific cellular pathways.</p>
						          
						              <a target="_blank" href="http://dx.doi.org/10.1016/j.exphem.2008.05.004 ">10.1016/j.exphem.2008.05.004 </a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 50  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading50">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse50" aria-expanded="true" aria-controls="collapse50">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">50. MicroRNA expression pattern of undifferentiated and differentiated human embryonic stem cells. (2007)</p>
						                 <p class="font-italic font-weight-light">Lakshmipathy U, Love B, <span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Jornsten R, Graichen R, Hart RP, Chesnut JD</p>
						                 <p>Stem Cells Dev. 2007 Dec;16(6):1003-16. doi: 10.1089/scd.2007.0026.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse50" class="collapse" aria-labelledby="heading50" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="18004940" class="altmetric-embed pull-left"> </div>
						          <p>Many of the currently established human embryonic stem (hES) cell lines have been characterized extensively in terms of their gene expression profiles and genetic stability in culture. Recent studies have indicated that microRNAs (miRNAs), a class of noncoding small RNAs that participate in the regulation of gene expression, may play a key role in stem cell self-renewal and differentiation. Using both microarrays and quantitative PCR, we report here the differences in miRNA expression between undifferentiated hES cells and their corresponding differentiated cells that underwent differentiation in vitro over a period of 2 weeks. Our results confirm the identity of a signature miRNA profile in pluripotent cells, comprising a small subset of differentially expressed miRNAs in hES cells. Examining both mRNA and miRNA profiles under multiple conditions using cross-correlation, we find clusters of miRNAs grouped with specific, biologically interpretable mRNAs. We identify patterns of expression in the progression from hES cells to differentiated cells that suggest a role for selected miRNAs in maintenance of the undifferentiated, pluripotent state. Profiling of the hES cell "miRNA-ome" provides an insight into molecules that control cellular differentiation and maintenance of the pluripotent state, findings that have broad implications in development, homeostasis, and human disease states.</p>
						          
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/18004940">Article</a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 51  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading51">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse51" aria-expanded="true" aria-controls="collapse51">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">51. Bioinformatic analysis of neural stem cell differentiation. (2007)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Davila J, Jornsten R, Keles S, Hart RP</p>
						                 <p>J Biomol Tech. 2007 Sep;18(4):205-12.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse51" class="collapse" aria-labelledby="heading51" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="17916793" class="altmetric-embed pull-left"> </div>
						          <p>Regulated mRnAs during differentiation of rat neural stem cells were analyzed using the ABi1700 microarray platform. This microarray, while technically advanced, suffers from the difficulty of integrating hybridization results into public databases for systems-level analysis. This is particularly true for the rat array, since many of the probes were designed for transcripts based on predicted human and mouse homologs. using several strategies, we increased the public annotation of the 27,531 probes from 43% to over 65%. To increase the dynamic range of annotation, probes were mapped to numerous public keys from several data sources. consensus annotation from multiple sources was determined for well-scoring alignments, and a confidence-based ranking system established for probes with less agreement across multiple data sources. previous attempts at genomic interpretation using the celera annotation model resulted in poor overlap with expected genomic sequences. since the public keys are more precisely mapped to the genome, we could now analyze the relationships between predicted transcription-factor binding sites and expression clusters. Results collected from a differentiation time course of two neural stem cell clones were clustered using a model-based algorithm. Transcription-factor binding sites were predicted from upstream regions of mapped transcripts using position weight matrices from either JAspAR or TRAnsFAc, and the resulting scores were used to discriminate between observed expression clusters. A classification and regression tree analysis was conducted using cluster numbers as gene identifiers and TFBs scores as predictors, pruning back to obtain a tree with the lowest gene class prediction error rate. Results identify several transcription factors, the presence or absence of which are sufficient to describe clusters of mRnAs changing over time-those that are static, as well as clusters describing cell line differences. public annotation of the AB1700 rat genome array will be valuable for integrating results into future systems-level analyses.</p>
						          
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17916793">Article</a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 52  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading52">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse52" aria-expanded="true" aria-controls="collapse52">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">52. Rational probe optimization and enhanced detection strategy for microRNAs using microarrays. (2005)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Yang M, Bowers J, Getts RC, Padgett RW, Hart RP</p>
						                 <p>RNA Biol. 2005 Jul-Sep;2(3):93-100. doi: 10.4161/rna.2.3.2059. Epub 2005 Jul 20.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse52" class="collapse" aria-labelledby="heading52" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="17114923" class="altmetric-embed pull-left"> </div>
						          <p>MicroRNAs (miRNAs) are post-transcriptional regulators participating in biological processes ranging from differentiation to carcinogenesis. We developed a rational probe design algorithm and a sensitive labelling scheme for optimizing miRNA microarrays. Our microarray contains probes for all validated miRNAs from five species, with the potential for drawing on species conservation to identify novel miRNAs with homologous probes. These methods are useful for high-throughput analysis of micro RNAs from various sources, and allow analysis with limiting quantities of RNA. The system design can also be extended for use on Luminex beads or on 96-well plates in an ELISA-style assay. We optimized hybridization temperatures using sequence variations on 20 of the probes and determined that all probes distinguish wild-type from 2 nt mutations, and most probes distinguish a 1 nt mutation, producing good selectivity between closely-related small RNA sequences. Results of tissue comparisons on our microarrays reveal patterns of hybridization that agree with results from Northern blots and other methods.</p>
						          
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17114923">Article</a>
						          
						        </div>
						      </div>
						    </div>
						    
								<!--  Card 53  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading53">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse53" aria-expanded="true" aria-controls="collapse53">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">53. Evaluation of sense-strand mRNA amplification by comparative quantitative PCR. (2004)</p>
						                 <p class="font-italic font-weight-light"><span class="font-weight-bold" style="font-size: 1.0rem"><u>Goff LA</u></span>, Bowers J, Schwalm J, Howerton K, Getts RC, Hart RP</p>
						                 <p>BMC Genomics. 2004 Oct 6;5:76. doi: 10.1186/1471-2164-5-76.</p>
												</span>
										</a>
						      </div>

						      <div id="collapse53" class="collapse" aria-labelledby="heading53" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="15469607" class="altmetric-embed pull-left"> </div>
						          <p>BACKGROUND: RNA amplification is required for incorporating laser-capture microdissection techniques into microarray assays. However, standard oligonucleotide microarrays contain sense-strand probes, so traditional T7 amplification schemes producing anti-sense RNA are not appropriate for hybridization when combined with conventional reverse transcription labeling methods. We wished to assess the accuracy of a new sense-strand RNA amplification method by comparing ratios between two samples using quantitative real-time PCR (qPCR), mimicking a two-color microarray assay. RESULTS: We performed our validation using qPCR. Three samples of rat brain RNA and three samples of rat liver RNA were amplified using several kits (Ambion messageAmp, NuGen Ovation, and several versions of Genisphere SenseAmp). Results were assessed by comparing the liver/brain ratio for 192 mRNAs before and after amplification. In general, all kits produced strong correlations with unamplified RNAs. The SenseAmp kit produced the highest correlation, and was also able to amplify a partially degraded sample accurately. CONCLUSION: We have validated an optimized sense-strand RNA amplification method for use in comparative studies such as two-color microarrays.</p>
						          
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/15469607">Article</a>
						          
						        </div>
						      </div>
						    </div>
						    
						  </div>
						    <!-- DA neuron preprint
						    <li>
						        <strong><em>Temporal and spatial variation among single dopaminergic neuron transcriptomes informs cellular phenotype diversity and Parkinson's Disease gene prioritization</em> (2017)</strong>
						                 <p class="note">Paul Hook, Sarah A. McClymont, Gabrielle H. Cannon, William D. Law, Loyal A. Goff<sup>*</sup>, Andrew S. McCallion<sup>*</sup><br>
						                 doi: <a href="https://doi.org/10.1101/148049" target="_blank">https://doi.org/10.1101/148049</a></br>
						          <p>Parkinson's Disease (PD) causes collapse of substantia nigra (SN) dopaminergic (DA) neurons of the midbrain (MB), while other DA populations are relatively spared. Here, we used single-cell RNA-seq (scRNA-seq) to characterize DA neuron populations in the mouse brain at embryonic and postnatal timepoints. These data allow for the discrimination between olfactory bulb (OB), forebrain (FB), and MB DA populations as well identification of subpopulations of DA neurons in each region. We observe a longitudinal axis of MB DA development, during which specialization and heterogeneity increases. We identify three distinct subpopulations of known MB DA neurons and provide evidence of a postnatal MB DA precursor, identifying novel markers for each subpopulation. Further, we discover gene regulatory networks (GRNs) that are significantly associated with neurodegenerative diseases and highly correlated with specific DA neuron subpopulations. By integrating these data with published genome-wide association studies (GWAS), we prioritize candidate genes in all 32 PD associated loci. Collectively, our data reveal genes and pathways that may begin to explain the selective vulnerability of SN DA neurons and allow for the systematic prioritization of genes in PD GWAS loci for functional evaluation.
						            <a target="_blank" href="http://biorxiv.org/content/early/2017/06/09/148049">Preprint</a><p>
								</li>
								-->
					</section>
			</div>
		</div>
</div>

		<!-- Footer -->
  <footer id="footer">
    <ul class="icons">
      <li><a href="https://twitter.com/intent/tweet?original_referer=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&ref_src=twsrc%5Etfw&text=Share%20Link%20Generator%3A%20Facebook%2C%20Twitter%2C%20LinkedIn%2C%20Pinterest%2C%20and%20Email%20mailto%20forms&tw_p=tweetbutton&url=https%3A%2F%2Fwww.sharelinkgenerator.com%2F&via=twitterapi" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
      <!-- <li><a href="#" class="icon brands alt fa-facebook-f"><span class="label">Facebook</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-linkedin-in"><span class="label">LinkedIn</span></a></li> -->
      <!-- <li><a href="#" class="icon brands alt fa-instagram"><span class="label">Instagram</span></a></li> -->
      <li><a href="http://github.com/gofflab/" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
      <li><a href="mailto:loyalgoff@jhmi.edu" class="icon solid alt fa-envelope"><span class="label">Email</span></a></li>
    </ul>
    <ul class="copyright">
      <li>&copy; Goff Lab - JHU. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
    </ul>
  </footer>
		<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/jquery.dropotron.min.js"></script>
<script src="assets/js/jquery.scrollex.min.js"></script>
<script src="assets/js/browser.min.js"></script>
<script src="assets/js/breakpoints.min.js"></script>
<script src="assets/js/util.js"></script>
<script src="assets/js/main.js"></script>
<script src="assets/js/jquery-ui.min.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script>
  $( function() {
    $( "#accordion" ).accordion();
  } );
  </script>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-318359-8', 'auto');
ga('send', 'pageview');
</script>
	</body>
</html>
