{% extends "base.html" %}

{% block title %}
	Publications
{% endblock title %}

{% block content %}
<body class="is-preload">
<div id="page-wrapper">
	{% include 'header.html' %}
	<!-- Main -->
		<div id="main" class="wrapper style1">
			<div class="container">
				<header class="major">
					<h2>Publications</h2>
						<!-- Publication summary links -->
						<p>
							<a href="http://europepmc.org/authors/0000-0003-2875-451X" target="_blank">EBI Summary page</a> |
							<a href="https://scholar.google.com/citations?user=2ZVv2v8AAAAJ&hl=en" target="_blank">Google Scholar</a> |
							<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Goff%2C+Loyal+%5BAuthor%5D+or+Goff%2C+Loyal+Andrew%5BAuthor%5D" target="_blank">Pubmed</a> |
							<a href="https://orcid.org/0000-0003-2875-451X" target="_blank">ORCHID: 0000-0003-2875-451X</a> |
							<a href="https://profiles.impactstory.org/u/0000-0003-2875-451X" target="_blank">Impactstory</a>
						</p>
				</header>
				<!-- Content -->
					<section id="content">

							<div class="accordion alt" id="accordionpubs">
						    {% for record in records %}
								<!--  Card {{ loop.index }}  -->
						    <div class="card text-left">
						      <div class="card-header" id="heading{{ loop.index }}">
										<a class="btn btn-dark pull-left" data-toggle="collapse" data-target="#collapse{{ loop.index }}" aria-expanded="true" aria-controls="collapse{{ loop.index }}">
												<span class="pull-left text-left">
														 <p class="font-weight-bold">{{ loop.index }}. {{ record['TI'] }} ({{ record['DP'][:4] }})</p>
						                 <p class="font-italic font-weight-light">{{ ", ".join(record['AU'])|nameBoldPubs }}</p>
						                 <p>{{ record['SO'].strip() }}</p>
												</span>
										</a>
						      </div>

						      <div id="collapse{{ loop.index }}" class="collapse" aria-labelledby="heading{{ loop.index }}" data-parent="#accordionpubs">
						        <div class="card-body" >
						          <div data-badge-type="1" data-hide-no-mentions="true" data-pmid="{{record['PMID']}}" class="altmetric-embed pull-left"> </div>
						          <p>{{ record['AB'] }}</p>
						          {% if record['LID'] is defined%}
						              <a target="_blank" href="http://dx.doi.org/{{ record['LID'].split('[doi]')[0] }}">{{ record['LID'].split('[doi]')[0] }}</a>
						          {% else %}
						              <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/{{ record['PMID'] }}">Article</a>
						          {% endif %}
						        </div>
						      </div>
						    </div>
						    {% endfor %}
						  </div>

						    <!-- DA neuron preprint
						    <li>
						        <strong><em>Temporal and spatial variation among single dopaminergic neuron transcriptomes informs cellular phenotype diversity and Parkinson's Disease gene prioritization</em> (2017)</strong>
						                 <p class="note">Paul Hook, Sarah A. McClymont, Gabrielle H. Cannon, William D. Law, Loyal A. Goff<sup>*</sup>, Andrew S. McCallion<sup>*</sup><br>
						                 doi: <a href="https://doi.org/10.1101/148049" target="_blank">https://doi.org/10.1101/148049</a></br>
						          <p>Parkinson's Disease (PD) causes collapse of substantia nigra (SN) dopaminergic (DA) neurons of the midbrain (MB), while other DA populations are relatively spared. Here, we used single-cell RNA-seq (scRNA-seq) to characterize DA neuron populations in the mouse brain at embryonic and postnatal timepoints. These data allow for the discrimination between olfactory bulb (OB), forebrain (FB), and MB DA populations as well identification of subpopulations of DA neurons in each region. We observe a longitudinal axis of MB DA development, during which specialization and heterogeneity increases. We identify three distinct subpopulations of known MB DA neurons and provide evidence of a postnatal MB DA precursor, identifying novel markers for each subpopulation. Further, we discover gene regulatory networks (GRNs) that are significantly associated with neurodegenerative diseases and highly correlated with specific DA neuron subpopulations. By integrating these data with published genome-wide association studies (GWAS), we prioritize candidate genes in all 32 PD associated loci. Collectively, our data reveal genes and pathways that may begin to explain the selective vulnerability of SN DA neurons and allow for the systematic prioritization of genes in PD GWAS loci for functional evaluation.
						            <a target="_blank" href="http://biorxiv.org/content/early/2017/06/09/148049">Preprint</a><p>
								</li>
								-->
					</section>
			</div>
		</div>
</div>
{% endblock content %}
